{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "#the raw file contains too many unnecessary lines. Remove all line-breaker, and add line-breaker only ahead of the start of a gene(containing ID) and localization info:\n",
    "\n",
    "f = open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/uniprot-organism%3Aescherichia-filtered-organism%3A_Escherichia+coli+%5B562%5D--.txt', 'r')\n",
    "\n",
    "f_str= f.read()\n",
    "# Check out the first 10 entries\n",
    "#f_list=f.readlines()\n",
    "f_str=f_str.replace('\\n','')\n",
    "f_str=f_str.replace('ID   ','\\n ID   ')\n",
    "f_str=f_str.replace('-!-','\\n -!-')\n",
    "\n",
    "#save the parsed file\n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_parsed.txt', 'w') as output:\n",
    "    output.write(f_str)\n",
    "#parsed = open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/K12_parsed.txt', 'r')\n",
    "#par_list= parsed.readlines()\n",
    "\n",
    "#par_list[:10]\n",
    "\n",
    "\n",
    "\n",
    "#From parsed file, slice lines starting with ID and subcellular location\n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_parsed.txt', 'r') as f, open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_IDlocal.txt', 'w') as f_out:\n",
    "    f.seek(0)\n",
    "    f_list = f.readlines()\n",
    "    for i,line in enumerate(f_list):\n",
    "        if f_list[i][:6]==' ID   ' and 'ECOLX' in line[:34]:\n",
    "            f_out.write(line[:19])\n",
    "            f_out.write('\\n')\n",
    "        if f_list[i][:25]==' -!- SUBCELLULAR LOCATION' and f_list[i+1][:25]!=' -!- SUBCELLULAR LOCATION':\n",
    "            f_list[i]=f_list[i].replace('\\n','')\n",
    "            f_out.write(f_list[i][4:400])\n",
    "            f_out.write('\\n')\n",
    "        \n",
    "  \n",
    "\n",
    "x = open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_IDlocal.txt', 'r')\n",
    "x_list = x.readlines()\n",
    "#x_list[:4]\n",
    "\n",
    "#check number of items in a list:\n",
    "#size = len(x_list)\n",
    "#size\n",
    "\n",
    "#add extra line for those lacking localization info:\n",
    "for i,j in enumerate(x_list):\n",
    "    if x_list[i][:6]==' ID   ' and x_list[i-1][:6]==' ID   ':\n",
    "        x_list.insert(i,'\\n')\n",
    "#x_list\n",
    "\n",
    "#export modified list into a file, need to convert list into string before writing into file:\n",
    "\n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_gene_localtion_everyotherline.txt', 'w') as output:\n",
    "    y=''.join(x_list)\n",
    "    output.write(y)\n",
    "\n",
    "    \n",
    "\n",
    "#remove the first line\n",
    "f = open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_gene_localtion_everyotherline.txt', 'r')\n",
    "\n",
    "f_new_str=f.read()\n",
    "f_new_str=f_new_str[1:]\n",
    "\n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_gene_localtion_everyotherline_newhead.txt', 'w') as output:\n",
    "    output.write(f_new_str)\n",
    "\n",
    "\n",
    "#keyword:    \n",
    "#nucleoid: 29\n",
    "#Cytoplasm 899\n",
    "##Cell membrane 260\n",
    "#Peripheral membrane protein 95\n",
    "#Periplasm.CC 62\n",
    "#Cell inner membrane 1010\n",
    "#Cell outer membrane 135\n",
    "#lipid-anchor 69\n",
    "#lipid-CC 25\n",
    "\n",
    "\n",
    "#extract ID and localization information and write into two separate .txt \n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_gene_localtion_everyotherline_newhead.txt', 'r') as f, open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_gene_final.txt', 'w') as f_gene, open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_local_final.txt', 'w') as f_local:\n",
    "    f.seek(0)\n",
    "    f_list = f.readlines()\n",
    "    gene_list=[]\n",
    "    local_list=[]\n",
    "    for i,line in enumerate(f_list):\n",
    "        if f_list[i][:6]==' ID   ':\n",
    "            f_gene.write(f_list[i][6:30])\n",
    "            \n",
    "            \n",
    "        if f_list[i][:1]=='\\n':\n",
    "            f_local.write('unclear')\n",
    "            f_local.write('\\n')\n",
    "        if f_list[i][:12]==' SUBCELLULAR':\n",
    "            key_word=''\n",
    "            if 'Cytoplasm 'in f_list[i][1:-1] or 'Cytoplasm.' in f_list[i][1:-1] or 'Cytoplasm,' in f_list[i][1:-1]:\n",
    "                key_word+='cyto;'\n",
    "            if 'Cell membrane' in f_list[i][1:-1] or 'Membrane ' in f_list[i][1:-1]:\n",
    "                key_word+='membrane;'\n",
    "            if 'Periplasm 'in f_list[i][1:-1] or 'Periplasm.' in f_list[i][1:-1] or 'Periplasm,' in f_list[i][1:-1]:\n",
    "                key_word+='Periplasm;'\n",
    "            if 'Cell inner membrane'in f_list[i][1:-1]:\n",
    "                key_word+='innermembrane;'\n",
    "            if 'Cell outer membrane'in f_list[i][1:-1]:\n",
    "                key_word+='Cell outer membrane;'\n",
    "            if 'Lipid-anchor'in f_list[i][1:-1]:\n",
    "                key_word+='lipid-anchor;'\n",
    "            if 'Lipid-CC'in f_list[i][1:-1]:\n",
    "                key_word+='lipid-CC;'\n",
    "            if 'Peripheral'in f_list[i][1:-1]:\n",
    "                key_word+='Peripheral_membrane_protein;'\n",
    "            if 'nucleoid'in f_list[i][1:-1]:\n",
    "                key_word+='nucleoid;'\n",
    "            if 'Secreted'in f_list[i][1:-1]:\n",
    "                key_word+='secreted;'\n",
    "            if 'Fimbrium'in f_list[i][1:-1]:\n",
    "                key_word+='Fimbrium;'\n",
    "            if 'flagellum'in f_list[i][1:-1]:\n",
    "                key_word+='flagellum;'\n",
    "            if 'host membrane'in f_list[i][1:-1]:\n",
    "                key_word+='host_membrane;'\n",
    "                \n",
    "            f_local.write(key_word)\n",
    "            f_local.write('\\n')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['\\n',\n",
       " ' ID   BLAT_ECOLX              Reviewed;         286 AA.AC   P62593; P00810; Q47313;DT   21-JUL-1986, integrated into UniProtKB/Swiss-Prot.DT   21-JUL-1986, sequence version 1.DT   25-APR-2018, entry version 113.DE   RecName: Full=Beta-lactamase TEM;DE            EC=3.5.2.6;DE   AltName: Full=IRT-4;DE   AltName: Full=Penicillinase;DE   AltName: Full=TEM-1;DE   AltName: Full=TEM-16/CAZ-7;DE   AltName: Full=TEM-2;DE   AltName: Full=TEM-24/CAZ-6;DE   AltName: Full=TEM-3;DE   AltName: Full=TEM-4;DE   AltName: Full=TEM-5;DE   AltName: Full=TEM-6;DE   AltName: Full=TEM-8/CAZ-2;DE   Flags: Precursor;GN   Name=bla;GN   andGN   Name=blaT-3;GN   andGN   Name=blaT-4;GN   andGN   Name=blaT-5;GN   andGN   Name=blaT-6;OS   Escherichia coli.OG   Plasmid R1 (R7268), Plasmid IncFII R100 (NR1), Plasmid R6K,OG   Plasmid pUD16, Plasmid pCFF04, and Plasmid pCFF14.OC   Bacteria; Proteobacteria; Gammaproteobacteria; Enterobacterales;OC   Enterobacteriaceae; Escherichia.OX   NCBI_TaxID=562;RN   [1]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-1).RC   PLASMID=R1 (R7268); TRANSPOSON=Tn3;RX   PubMed=358200; DOI=10.1073/pnas.75.8.3737;RA   Sutcliffe J.G.;RT   \"Nucleotide sequence of the ampicillin resistance gene of EscherichiaRT   coli plasmid pBR322.\";RL   Proc. Natl. Acad. Sci. U.S.A. 75:3737-3741(1978).RN   [2]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-1).RC   PLASMID=R1 (R7268); TRANSPOSON=Tn3;RX   PubMed=383387;RA   Sutcliffe J.G.;RT   \"Complete nucleotide sequence of the Escherichia coli plasmidRT   pBR322.\";RL   Cold Spring Harb. Symp. Quant. Biol. 43:77-90(1979).RN   [3]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-1).RC   PLASMID=IncFII R100 (NR1);RX   PubMed=3019092; DOI=10.1016/0065-227X(86)90018-3;RA   Ohtsubo H., Ryder T.B., Maeda Y., Armstrong K., Ohtsubo E.;RT   \"DNA replication of the resistance plasmid R100 and its control.\";RL   Adv. Biophys. 21:115-133(1986).RN   [4]RP   PROTEIN SEQUENCE OF 24-286 (TEM-2).RC   PLASMID=R6K; TRANSPOSON=Tn1;RX   PubMed=358199; DOI=10.1073/pnas.75.8.3732;RA   Ambler R.P., Scott G.K.;RT   \"Partial amino acid sequence of penicillinase coded by EscherichiaRT   coli plasmid R6K.\";RL   Proc. Natl. Acad. Sci. U.S.A. 75:3732-3736(1978).RN   [5]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-3).RA   Sougakoff W., Goussard S., Courvalin P.;RT   \"The TEM-3 beta-lactamase, which hydrolyzes broad-spectrumRT   cephalosporins, is derived from the TEM-2 penicillinase by two aminoRT   acid substitutions.\";RL   FEMS Microbiol. Lett. 56:343-348(1988).RN   [6]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-3).RC   PLASMID=pCFF04;RX   PubMed=1331747; DOI=10.1007/BF00286188;RA   Mabilat C., Lourencao-Vital J., Goussard S., Courvalin P.;RT   \"A new example of physical linkage between Tn1 and Tn21: theRT   antibiotic multiple-resistance region of plasmid pCFF04 encodingRT   extended-spectrum beta-lactamase TEM-3.\";RL   Mol. Gen. Genet. 235:113-121(1992).RN   [7]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-4 AND TEM-5).RC   STRAIN=CB86134; PLASMID=pCFF04, and pUD16;RX   PubMed=2550326; DOI=10.1016/0378-1119(89)90236-9;RA   Sougakoff W., Petit A., Goussard S., Sirot D., Bure A., Courvalin P.;RT   \"Characterization of the plasmid genes blaT-4 and blaT-5 which encodeRT   the broad-spectrum beta-lactamases TEM-4 and TEM-5 inRT   enterobacteriaceae.\";RL   Gene 78:339-348(1989).RN   [8]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-6).RC   STRAIN=HB251;RX   PubMed=1665171; DOI=10.1099/00221287-137-12-2681;RA   Goussard S., Sougakoff W., Mabilat C., Bauernfeind A., Courvalin P.;RT   \"An IS1-like element is responsible for high-level synthesis ofRT   extended-spectrum beta-lactamase TEM-6 in Enterobacteriaceae.\";RL   J. Gen. Microbiol. 137:2681-2687(1991).RN   [9]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA] (TEM-8; TEM-16 AND TEM-24).RX   PubMed=1416873; DOI=10.1128/AAC.36.9.1817;RA   Chanal C., Poupart M.C., Sirot D., Labia R., Sirot J., Cluzel R.;RT   \"Nucleotide sequences of CAZ-2, CAZ-6, and CAZ-7 beta-lactamaseRT   genes.\";RL   Antimicrob. Agents Chemother. 36:1817-1820(1992).RN   [10]RP   PROTEIN SEQUENCE OF 24-286 (IRT-4).RC   STRAIN=PEY;RX   PubMed=8056282; DOI=10.1111/j.1574-6968.1994.tb07016.x;RA   Brun T., Peduzzi J., Canica M.M., Paul G., Nevot P., Barthelemy M.,RA   Labia R.;RT   \"Characterization and amino acid sequence of IRT-4, a novel TEM-typeRT   enzyme with a decreased susceptibility to beta-lactamase inhibitors.\";RL   FEMS Microbiol. Lett. 120:111-117(1994).RN   [11]RP   X-RAY CRYSTALLOGRAPHY (2.5 ANGSTROMS) OF TEM-1.RX   PubMed=1544485; DOI=10.1016/0014-5793(92)80232-6;RA   Jelsch C., Lenfant F., Masson J.-M., Samama J.-P.;RT   \"Beta-lactamase TEM1 of E. coli. Crystal structure determination atRT   2.5-A resolution.\";RL   FEBS Lett. 299:135-142(1992).RN   [12]RP   X-RAY CRYSTALLOGRAPHY (1.8 ANGSTROMS) OF TEM-1.RX   PubMed=8356032; DOI=10.1002/prot.340160406;RA   Jelsch C., Mourey L., Masson J.-M., Samama J.-P.;RT   \"Crystal structure of Escherichia coli TEM1 beta-lactamase at 1.8-ART   resolution.\";RL   Proteins 16:364-383(1993).RN   [13]RP   X-RAY CRYSTALLOGRAPHY (1.7 ANGSTROMS) OF TEM-1 COMPLEXED WITH BLIP.RX   PubMed=8605632; DOI=10.1038/nsb0396-290;RA   Strynadka N.C.J., Jensen S.E., Alzari P.M., James M.N.G.;RT   \"A potent new mode of beta-lactamase inhibition revealed by the 1.7 ART   X-ray crystallographic structure of the TEM-1-BLIP complex.\";RL   Nat. Struct. Biol. 3:290-297(1996).RN   [14]RP   X-RAY CRYSTALLOGRAPHY (2.0 ANGSTROMS) OF TEM-1.RX   PubMed=9485412; DOI=10.1021/bi972501b;RA   Maveyraud L., Pratt R.F., Samama J.-P.;RT   \"Crystal structure of an acylation transition-state analog of the TEM-RT   1 beta-lactamase. Mechanistic implications for class A beta-RT   lactamases.\";RL   Biochemistry 37:2622-2628(1998).RN   [15]RP   X-RAY CRYSTALLOGRAPHY (2.28 ANGSTROMS) OF TEM-1.RX   PubMed=10423234; DOI=10.1021/bi990758z;RA   Swaren P., Golemi D., Cabantous S., Bulychev A., Maveyraud L.,RA   Mobashery S., Samama J.-P.;RT   \"X-ray structure of the Asn276Asp variant of the Escherichia coli TEM-RT   1 beta-lactamase: direct observation of electrostatic modulation inRT   resistance to inactivation by clavulanic acid.\";RL   Biochemistry 38:9570-9576(1999).CC   \\n',\n",
       " ' -!- FUNCTION: TEM-type are the most prevalent beta-lactamases inCC       enterobacteria; they hydrolyze the beta-lactam bond in susceptibleCC       beta-lactam antibiotics, thus conferring resistance to penicillinsCC       and cephalosporins. TEM-3 and TEM-4 are capable of hydrolyzingCC       cefotaxime and ceftazidime. TEM-5 is capable of hydrolyzingCC       ceftazidime. TEM-6 is capable of hydrolyzing ceftazidime andCC       aztreonam. TEM-8/CAZ-2, TEM-16/CAZ-7 and TEM-24/CAZ-6 are markedlyCC       active against ceftazidime. IRT-4 shows resistance to beta-CC       lactamase inhibitors.CC   \\n']"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#From parsed file, slice lines starting with ID and subcellular location\n",
    "f= open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/ECOLX_parsed.txt', 'r')\n",
    "f.seek(0)\n",
    "f_list = f.readlines()\n",
    "f_list[:3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
