{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['\\n',\n",
       " ' ID   MAZF_ECOLI              Reviewed;         111 AA.AC   P0AE70; P33645; Q2MA50;DT   06-DEC-2005, integrated into UniProtKB/Swiss-Prot.DT   06-DEC-2005, sequence version 1.DT   25-APR-2018, entry version 98.DE   RecName: Full=Endoribonuclease toxin MazF;DE            EC=3.1.27.- {ECO:0000269|PubMed:15537630};DE   AltName: Full=Toxin MazF;DE   AltName: Full=mRNA interferase MazF {ECO:0000303|PubMed:15537630};GN   Name=mazF; Synonyms=chpA, chpAK; OrderedLocusNames=b2782, JW2753;OS   Escherichia coli (strain K12).OC   Bacteria; Proteobacteria; Gammaproteobacteria; Enterobacterales;OC   Enterobacteriaceae; Escherichia.OX   NCBI_TaxID=83333;RN   [1]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA], AND OPERON STRUCTURE.RC   STRAIN=K12;RX   PubMed=2844820;RA   Metzger S., Dror I.B., Aizenman E., Schreiber G., Toone M.,RA   Friesen J.D., Cashel M., Glaser G.;RT   \"The nucleotide sequence and characterization of the relA gene ofRT   Escherichia coli.\";RL   J. Biol. Chem. 263:15699-15704(1988).RN   [2]RP   NUCLEOTIDE SEQUENCE [GENOMIC DNA].RC   STRAIN=K12 / MC1000 / ATCC 39531;RX   PubMed=8226627; DOI=10.1128/jb.175.21.6850-6856.1993;RA   Masuda Y., Miyakawa K., Nishimura Y., Ohtsubo E.;RT   \"chpA and chpB, Escherichia coli chromosomal homologs of the pem locusRT   responsible for stable maintenance of plasmid R100.\";RL   J. Bacteriol. 175:6850-6856(1993).RN   [3]RP   NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA].RC   STRAIN=K12 / MG1655 / ATCC 47076;RX   PubMed=9278503; DOI=10.1126/science.277.5331.1453;RA   Blattner F.R., Plunkett G. III, Bloch C.A., Perna N.T., Burland V.,RA   Riley M., Collado-Vides J., Glasner J.D., Rode C.K., Mayhew G.F.,RA   Gregor J., Davis N.W., Kirkpatrick H.A., Goeden M.A., Rose D.J.,RA   Mau B., Shao Y.;RT   \"The complete genome sequence of Escherichia coli K-12.\";RL   Science 277:1453-1462(1997).RN   [4]RP   NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA].RC   STRAIN=K12 / W3110 / ATCC 27325 / DSM 5911;RX   PubMed=16738553; DOI=10.1038/msb4100049;RA   Hayashi K., Morooka N., Yamamoto Y., Fujita K., Isono K., Choi S.,RA   Ohtsubo E., Baba T., Wanner B.L., Mori H., Horiuchi T.;RT   \"Highly accurate genome sequences of Escherichia coli K-12 strainsRT   MG1655 and W3110.\";RL   Mol. Syst. Biol. 2:E1-E5(2006).RN   [5]RP   FUNCTION IN PROGRAMMED CELL DEATH, INTERACTION WITH MAZE, INDUCTION,RP   DISRUPTION PHENOTYPE, AND OPERON STRUCTURE.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=8650219; DOI=10.1073/pnas.93.12.6059;RA   Aizenman E., Engelberg-Kulka H., Glaser G.;RT   \"An Escherichia coli chromosomal \\'addiction module\\' regulated byRT   guanosine 3\\',5\\'-bispyrophosphate: a model for programmed bacterialRT   cell death.\";RL   Proc. Natl. Acad. Sci. U.S.A. 93:6059-6063(1996).RN   [6]RP   TRANSCRIPTION REGULATION, AND SUBUNIT.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=11071896; DOI=10.1074/jbc.M008832200;RA   Marianovsky I., Aizenman E., Engelberg-Kulka H., Glaser G.;RT   \"The regulation of the Escherichia coli mazEF promoter involves anRT   unusual alternating palindrome.\";RL   J. Biol. Chem. 276:5975-5984(2001).RN   [7]RP   FUNCTION IN CELL DEATH, AND DISRUPTION PHENOTYPE.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=11222603; DOI=10.1128/JB.183.6.2041-2045.2001;RA   Sat B., Hazan R., Fisher T., Khaner H., Glaser G., Engelberg-Kulka H.;RT   \"Programmed cell death in Escherichia coli: some antibiotics canRT   trigger mazEF lethality.\";RL   J. Bacteriol. 183:2041-2045(2001).RN   [8]RP   FUNCTION AS A TOXIN.RC   STRAIN=K12;RX   PubMed=12123459; DOI=10.1046/j.1365-2958.2002.03027.x;RA   Pedersen K., Christensen S.K., Gerdes K.;RT   \"Rapid induction and reversal of a bacteriostatic condition byRT   controlled expression of toxins and antitoxins.\";RL   Mol. Microbiol. 45:501-510(2002).RN   [9]RP   RNA CLEAVAGE, INDUCTION, AND DISRUPTION PHENOTYPE.RC   STRAIN=K12 / MG1655 / ATCC 47076;RX   PubMed=12972253; DOI=10.1016/S0022-2836(03)00922-7;RA   Christensen S.K., Pedersen K., Hansen F.G., Gerdes K.;RT   \"Toxin-antitoxin loci as stress-response-elements: ChpAK/MazF andRT   ChpBK cleave translated RNAs and are counteracted by tmRNA.\";RL   J. Mol. Biol. 332:809-819(2003).RN   [10]RP   DISRUPTION PHENOTYPE, FUNCTION IN STRESS RESPONSE, AND DEPENDENCE ONRP   RELA.RC   STRAIN=K12 / K38 / S26, and K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=15150257; DOI=10.1128/JB.186.11.3663-3669.2004;RA   Hazan R., Sat B., Engelberg-Kulka H.;RT   \"Escherichia coli mazEF-mediated cell death is triggered by variousRT   stressful conditions.\";RL   J. Bacteriol. 186:3663-3669(2004).RN   [11]RP   FUNCTION IN PROGRAMMED CELL DEATH.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=15576778; DOI=10.1128/JB.186.24.8295-8300.2004;RA   Amitai S., Yassin Y., Engelberg-Kulka H.;RT   \"MazF-mediated cell death in Escherichia coli: a point of no return.\";RL   J. Bacteriol. 186:8295-8300(2004).RN   [12]RP   FUNCTION IN PHAGE IMMUNITY, AND DISRUPTION PHENOTYPE.RC   STRAIN=K12 / JM109 / ATCC 53323, K12 / K38 / S26, andRC   K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=15316771; DOI=10.1007/s00438-004-1048-y;RA   Hazan R., Engelberg-Kulka H.;RT   \"Escherichia coli mazEF-mediated cell death as a defense mechanismRT   that inhibits the spread of phage P1.\";RL   Mol. Genet. Genomics 272:227-234(2004).RN   [13]RP   FUNCTION AS AN ENDONUCLEASE, ENZYME REGULATION, SUBSTRATE SPECIFICITY,RP   AND RNA-BINDING.RX   PubMed=15537630; DOI=10.1074/jbc.M411811200;RA   Zhang Y., Zhang J., Hara H., Kato I., Inouye M.;RT   \"Insights into the mRNA cleavage mechanism by MazF, an mRNART   interferase.\";RL   J. Biol. Chem. 280:3143-3150(2005).RN   [14]RP   BIOTECHNOLOGY.RX   PubMed=15837428; DOI=10.1016/j.molcel.2005.03.011;RA   Suzuki M., Zhang J., Liu M., Woychik N.A., Inouye M.;RT   \"Single protein production in living cells facilitated by an mRNART   interferase.\";RL   Mol. Cell 18:253-261(2005).RN   [15]RP   INDUCTION, AND OPERON STRUCTURE.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574, andRC   K12 / W3110 / ATCC 27325 / DSM 5911;RX   PubMed=16390452; DOI=10.1111/j.1365-2958.2005.04956.x;RA   Gross M., Marianovsky I., Glaser G.;RT   \"MazG -- a regulator of programmed cell death in Escherichia coli.\";RL   Mol. Microbiol. 59:590-601(2006).RN   [16]RP   REQUIREMENT FOR EXTRACELLULAR DEATH FACTOR (EDF).RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=17962566; DOI=10.1126/science.1147248;RA   Kolodkin-Gal I., Hazan R., Gaathon A., Carmeli S., Engelberg-Kulka H.;RT   \"A linear pentapeptide is a quorum-sensing factor required for mazEF-RT   mediated cell death in Escherichia coli.\";RL   Science 318:652-655(2007).RN   [17]RP   EDF PRODUCTION, AND STRAIN DIFFERENCES IN ABILITY TO MAKE AND RESPONDRP   TO EDF.RC   STRAIN=K12 / K38 / S26, K12 / MC4100 / ATCC 35695 / DSM 6574,RC   K12 / MG1655 / ATCC 47076, and K12 / W3110 / ATCC 27325 / DSM 5911;RX   PubMed=18310334; DOI=10.1128/JB.01918-07;RA   Kolodkin-Gal I., Engelberg-Kulka H.;RT   \"The extracellular death factor: physiological and genetic factorsRT   influencing its production and response in Escherichia coli.\";RL   J. Bacteriol. 190:3169-3175(2008).RN   [18]RP   TRANSLATION IMPROVES CLEAVAGE EFFICIENCY.RC   STRAIN=K12;RX   PubMed=18854355; DOI=10.1093/nar/gkn667;RA   Christensen-Dalsgaard M., Gerdes K.;RT   \"Translation affects YoeB and MazF messenger RNA interferaseRT   activities by different mechanisms.\";RL   Nucleic Acids Res. 36:6472-6481(2008).RN   [19]RP   FUNCTION.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=19251848; DOI=10.1128/JB.00011-09;RA   Kolodkin-Gal I., Engelberg-Kulka H.;RT   \"The stationary-phase sigma factor sigma(S) is responsible for theRT   resistance of Escherichia coli stationary-phase cells to mazEF-RT   mediated cell death.\";RL   J. Bacteriol. 191:3177-3182(2009).RN   [20]RP   FUNCTION IN CELL DEATH, FUNCTION IN BIOFILM FORMATION, AND DISRUPTIONRP   PHENOTYPE.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=19707553; DOI=10.1371/journal.pone.0006785;RA   Kolodkin-Gal I., Verdiger R., Shlosberg-Fedida A., Engelberg-Kulka H.;RT   \"A differential effect of E. coli toxin-antitoxin systems on cellRT   death in liquid media and biofilm formation.\";RL   PLoS ONE 4:E6785-E6785(2009).RN   [21]RP   FUNCTION, SUBSTRATE SPECIFICITY, AND TRANSLATION REGULATION.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=21944167; DOI=10.1016/j.cell.2011.07.047;RA   Vesper O., Amitai S., Belitsky M., Byrgazov K., Kaberdina A.C.,RA   Engelberg-Kulka H., Moll I.;RT   \"Selective translation of leaderless mRNAs by specialized ribosomesRT   generated by MazF in Escherichia coli.\";RL   Cell 147:147-157(2011).RN   [22]RP   FUNCTION, ENZYME REGULATION, INTERACTION WITH EDF, AND SUBUNIT.RX   PubMed=21419338; DOI=10.1016/j.molcel.2011.02.023;RA   Belitsky M., Avshalom H., Erental A., Yelin I., Kumar S., London N.,RA   Sperber M., Schueler-Furman O., Engelberg-Kulka H.;RT   \"The Escherichia coli extracellular death factor EDF induces theRT   endoribonucleolytic activities of the toxins MazF and ChpBK.\";RL   Mol. Cell 41:625-635(2011).RN   [23]RP   FUNCTION IN PERSISTENCE, AND DISRUPTION PHENOTYPE.RC   STRAIN=K12 / MG1655 / ATCC 47076;RX   PubMed=21788497; DOI=10.1073/pnas.1100186108;RA   Maisonneuve E., Shakespeare L.J., Joergensen M.G., Gerdes K.;RT   \"Bacterial persistence by RNA endonucleases.\";RL   Proc. Natl. Acad. Sci. U.S.A. 108:13206-13211(2011).RN   [24]RP   FUNCTION, AND DISRUPTION PHENOTYPE.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=20005847; DOI=10.1016/j.molcel.2009.11.024;RA   Davies B.W., Kohanski M.A., Simmons L.A., Winkler J.A., Collins J.J.,RA   Walker G.C.;RT   \"Hydroxyurea induces hydroxyl radical-mediated cell death inRT   Escherichia coli.\";RL   Mol. Cell 36:845-860(2009).RN   [25]RP   FUNCTION, AND DISRUPTION PHENOTYPE.RC   STRAIN=K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=22412352; DOI=10.1371/journal.pbio.1001281;RA   Erental A., Sharon I., Engelberg-Kulka H.;RT   \"Two programmed cell death systems in Escherichia coli: an apoptotic-RT   like death is inhibited by the mazEF-mediated death pathway.\";RL   PLoS Biol. 10:E1001281-E1001281(2012).RN   [26]RP   FUNCTION, AND INDUCTION BY OTHER TA SYSTEMS.RC   STRAIN=K12 / BW25113;RX   PubMed=23432955; DOI=10.1186/1471-2180-13-45;RA   Kasari V., Mets T., Tenson T., Kaldalu N.;RT   \"Transcriptional cross-activation between toxin-antitoxin systems ofRT   Escherichia coli.\";RL   BMC Microbiol. 13:45-45(2013).RN   [27]RP   FUNCTION, ENZYME REGULATION, AND DISRUPTION PHENOTYPE.RC   STRAIN=K12 / MG1655 / ATCC 47076;RX   PubMed=23416055; DOI=10.1016/j.celrep.2013.01.026;RA   Dorsey-Oresto A., Lu T., Mosel M., Wang X., Salz T., Drlica K.,RA   Zhao X.;RT   \"YihE kinase is a central regulator of programmed cell death inRT   bacteria.\";RL   Cell Rep. 3:528-537(2013).RN   [28]RP   FUNCTION, SUBSTRATE SPECIFICITY, DOMAIN, AND MUTAGENESIS OFRP   17-PHE--HIS-28 AND 53-THR--GLU-61.RC   STRAIN=K12 / BW25113;RX   PubMed=23280569; DOI=10.1002/prot.24246;RA   Park J.H., Yoshizumi S., Yamaguchi Y., Wu K.P., Inouye M.;RT   \"ACA-specific RNA sequence recognition is acquired via the loop 2RT   region of MazF mRNA interferase.\";RL   Proteins 81:874-883(2013).RN   [29]RP   FUNCTION IN PERSISTENCE, DISRUPTION PHENOTYPE, AND MUTAGENESIS OFRP   GLU-24 AND HIS-28.RC   STRAIN=K12 / BW25113, and K12 / MC4100 / ATCC 35695 / DSM 6574;RX   PubMed=24375411; DOI=10.1074/jbc.M113.510511;RA   Tripathi A., Dewan P.C., Siddique S.A., Varadarajan R.;RT   \"MazF-induced growth inhibition and persister generation inRT   Escherichia coli.\";RL   J. Biol. Chem. 289:4191-4205(2014).RN   [30]RP   FUNCTION IN TRANSCRIPTION, AND SUBUNIT.RX   PubMed=25564525; DOI=10.1093/nar/gku1352;RA   Zorzini V., Buts L., Schrank E., Sterckx Y.G., Respondek M.,RA   Engelberg-Kulka H., Loris R., Zangger K., van Nuland N.A.;RT   \"Escherichia coli antitoxin MazE as transcription factor: insightsRT   into MazE-DNA binding.\";RL   Nucleic Acids Res. 43:1241-1256(2015).RN   [31]RP   REVIEW.RX   PubMed=19215780; DOI=10.1016/S0079-6603(08)00812-X;RA   Yamaguchi Y., Inouye M.;RT   \"mRNA interferases, sequence-specific endoribonucleases from theRT   toxin-antitoxin systems.\";RL   Prog. Mol. Biol. Transl. Sci. 85:467-500(2009).RN   [32]RP   X-RAY CRYSTALLOGRAPHY (1.7 ANGSTROMS) OF 2-111, POSSIBLE DNA-BINDING,RP   AND SUBUNIT.RX   PubMed=12718874; DOI=10.1016/S1097-2765(03)00097-2;RA   Kamada K., Hanaoka F., Burley S.K.;RT   \"Crystal structure of the MazE/MazF complex: molecular bases ofRT   antidote-toxin recognition.\";RL   Mol. Cell 11:875-884(2003).RN   [33]RP   STRUCTURE BY NMR, AND MUTAGENESIS OF GLU-24.RX   PubMed=16413577; DOI=10.1016/j.jmb.2005.12.035;RA   Li G.Y., Zhang Y., Chan M.C., Mal T.K., Hoeflich K.P., Inouye M.,RA   Ikura M.;RT   \"Characterization of dual substrate binding sites in the homodimericRT   structure of Escherichia coli mRNA interferase MazF.\";RL   J. Mol. Biol. 357:139-150(2006).RN   [34]RP   X-RAY CRYSTALLOGRAPHY (2.00 ANGSTROMS).RA   Wang X., Wang K., Gao X., Zhang X., Li L., Su X., Zhang J.;RT   \"Biochemical and structural analysis of E. coli MazF toxin.\";RL   Submitted (JUN-2010) to the PDB data bank.CC   \\n',\n",
       " ' -!- FUNCTION: Toxic component of a type II toxin-antitoxin (TA)CC       system. A sequence-specific endoribonuclease it inhibits proteinCC       synthesis by cleaving mRNA and inducing bacterial stasis. It isCC       stable, single-strand specific with mRNA cleavage independent ofCC       the ribosome, although translation enhances cleavage for someCC       mRNAs (PubMed:18854355). Cleavage occurs at the 5\\'-end of ACACC       sequences, yielding a 2\\',3\\'-cyclic phosphate and a free 5\\'-OH,CC       although cleavage can also occur on the 3\\'-end of the first ACC       (PubMed:15537630, PubMed:23280569). Digests 16S rRNA in vivo 43CC       nts upstream of the C-terminus; this removes the anti-Shine-CC       Dalgarno sequence forming a mixed population of wild-type andCC       \"stress ribosomes\". Stress ribosomes do not translate leader-CC       containing mRNA but are proficient in translation of leaderlessCC       mRNA, which alters the protein expression profile of the cell;CC       MazF produces some leaderless mRNA (PubMed:21944167). The toxicCC       endoribonuclease activity is inhibited by its labile cognateCC       antitoxin MazE. Toxicity results when the levels of MazE decreaseCC       in the cell, leading to mRNA degradation. This effect can beCC       rescued by expression of MazE, but after 6 hours in rich mediumCC       overexpression of MazF leads to programmed cell deathCC       (PubMed:8650219, PubMed:11222603). MazF-mediated cell death occursCC       following a number of stress conditions in a relA-dependentCC       fashion and only when cells are in log phase; sigma factor SCC       (rpoS) protects stationary phase cells from MazF-killingCC       (PubMed:15150257, PubMed:19251848). Cell growth and viability areCC       not affected when MazF and MazE are coexpressed. Both MazE andCC       MazE-MazF bind to the promoter region of the mazE-mazF operon toCC       inhibit their own transcription. MazE has higher affinity forCC       promoter DNA in the presence of MazF (PubMed:25564525). Cross-talkCC       can occur between different TA systems, ectopic expression of thisCC       toxin induces transcription of the relBEF TA system operon withCC       specific cleavage of the mRNA produced (PubMed:23432955).CC       {ECO:0000269|PubMed:11071896, ECO:0000269|PubMed:11222603,CC       ECO:0000269|PubMed:15150257, ECO:0000269|PubMed:15537630,CC       ECO:0000269|PubMed:18854355, ECO:0000269|PubMed:19251848,CC       ECO:0000269|PubMed:21944167, ECO:0000269|PubMed:23280569,CC       ECO:0000269|PubMed:23432955, ECO:0000269|PubMed:25564525,CC       ECO:0000269|PubMed:8650219}.CC   \\n',\n",
       " ' -!- FUNCTION: Might also serve to protect cells against bacteriophage;CC       in the presence of MazE-MazF fewer P1 phage are produced than in aCC       disrupted strain. For strain K38 most wild-type cells are killedCC       but not by phage lysis; it was suggested that MazE-MazF causes P1CC       phage exclusion from the bacterial population. This phenomenon isCC       strain dependent. {ECO:0000269|PubMed:15316771}.CC   \\n',\n",
       " ' -!- FUNCTION: The physiological role of this TA system is debated.CC       Programmed cell death (PCD) occurs when cells are at high densityCC       and depends on the presence of MazE-MazF and a quorum sensingCC       pentapeptide, the extracellular death factor (EDF) with sequenceCC       Asn-Asn-Trp-Asn-Asn (NNWNN), probably produced from the zwf geneCC       product glucose-6-phosphate 1-dehydrogenase (PubMed:17962566,CC       PubMed:18310334). Cell death governed by the MazE-MazF and DinJ-CC       YafQ TA systems seems to play a role in biofilm formation, whileCC       MazE-MazF is also implicated in cell death in liquid mediaCC       (PubMed:19707553). Implicated in hydroxy radical-mediated cellCC       death induced by hydroxyurea treatment (PubMed:20005847,CC       PubMed:23416055). In conjunction with EDF prevents apoptotic-likeCC       death (ALD) in the presence of DNA damaging agents, probably byCC       reducing recA mRNA levels in a non-endonuclease-mediated mannerCC       (PubMed:22412352). Other studies (in strains BW25113 and MC4100,CC       the latter makes EDF) demonstrate MazF does not cause PCD butCC       instead bacteriostasis and possibly a dormant state as well asCC       persister cell generation (PubMed:24375411). mRNA interferasesCC       play a role in bacterial persistence to antibiotics;CC       overexpression of this protein induces persisters resistant toCC       ciprofloxacin and ampicillin (PubMed:21788497).CC       {ECO:0000269|PubMed:17962566, ECO:0000269|PubMed:18310334,CC       ECO:0000269|PubMed:19707553, ECO:0000269|PubMed:20005847,CC       ECO:0000269|PubMed:21419338, ECO:0000269|PubMed:21788497,CC       ECO:0000269|PubMed:22412352, ECO:0000269|PubMed:23416055,CC       ECO:0000269|PubMed:24375411, ECO:0000269|PubMed:8650219}.CC   \\n',\n",
       " ' -!- ENZYME REGULATION: Inhibited by Mg(2+) (PubMed:15537630).CC       Stimulated in vitro in a concentration-dependent fashion by EDF,CC       which is able to overcome inhibition by cognate antitoxin MazECC       (PubMed:21419338). The TA system is antagonized by stress responseCC       kinase SrkA, but probably not by phosphorylation of MazFCC       (PubMed:23416055). {ECO:0000269|PubMed:15537630,CC       ECO:0000269|PubMed:21419338, ECO:0000269|PubMed:23416055}.CC   \\n',\n",
       " ' -!- SUBUNIT: Probably a dimer. Forms a heterohexamer composed ofCC       alternating toxin and antitoxin homodimers MazF(2)-MazE(2)-CC       MazF(2). The binding site of MazE and ssRNA or ssDNA are largelyCC       overlapping; the presence of only 1 MazE molecule inhibits mRNACC       endoribonuclease activity. Binds to EDF but not a mutated EDFCC       (NNGNN) (PubMed:21419338). {ECO:0000269|PubMed:11071896,CC       ECO:0000269|PubMed:12718874, ECO:0000269|PubMed:21419338,CC       ECO:0000269|PubMed:25564525, ECO:0000269|PubMed:8650219}.CC   \\n',\n",
       " ' -!- INDUCTION: Expressed in exponentially growing cells. Induction hasCC       been reported to occur after amino acid starvation in a ppGpp-CC       independent fashion and to be Lon protease-dependentCC       (PubMed:12972253), but also to not occur after amino acidCC       starvation and to be regulated by ppGpp (PubMed:8650219). AlsoCC       induced in M9 minimal medium and by chloramphenicol treatmentCC       (PubMed:21944167). MazE alone and in combination with MazF,CC       represses transcription of the mazE-mazF operon. Fis activatesCC       transcription. Part of the relA-mazE-mazF-mazG operon, there isCC       also a second mazE-mazF specific promoter which is negativelyCC       autoregulated (PubMed:2844820, PubMed:8650219). Operon induced byCC       ectopic expression of toxin RelE; operon induction by amino acidCC       starvation requires the relBEF operon (PubMed:23432955).CC       {ECO:0000269|PubMed:11071896, ECO:0000269|PubMed:12972253,CC       ECO:0000269|PubMed:16390452, ECO:0000269|PubMed:21944167,CC       ECO:0000269|PubMed:23432955, ECO:0000269|PubMed:2844820,CC       ECO:0000269|PubMed:8650219}.CC   \\n',\n",
       " ' -!- DOMAIN: Loop 1 (residues 17-28) effects catalytic activity whileCC       recognition of the ACA cleavage site is influenced by loop 2CC       (residues 53-61). Alterations of loop 2 generate new cleavageCC       sites in addition to retaining the original cleavage site.CC       {ECO:0000305|PubMed:23280569}.CC   \\n',\n",
       " ' -!- DISRUPTION PHENOTYPE: Decreased sensitivity to dramaticCC       intracellular increases of ppGpp. Cells missing mazE-mazF surviveCC       high temperature, various DNA-damaging agents and H(2)O(2)CC       exposure better than wild-type cells. Cells missing mazE-mazFCC       produce more P1 phage than wild-type cells, while introduction ofCC       lysogens into a growing non-lysogenic disruption culture is lethalCC       (PubMed:15316771). Cells missing mazE-mazF show reduced biofilmCC       formation, and survive antibiotic treatment in log phase betterCC       than wild-type cells (PubMed:11222603, PubMed:19707553). HoweverCC       lag phase cells disrupted only for mazF had a lower survival rateCC       than wild-type cells (PubMed:24375411). Cells missing mazE-mazFCC       survive hydroxyurea treatment better than wild-type; furtherCC       disruption of relE-relB and tonB yields even better survivalCC       (PubMed:20005847). Cells missing mazE-mazF undergo an apoptotic-CC       like death (ALD) upon DNA damage characterized by membraneCC       depolarization and DNA fragmentation; further disruption of recACC       prevents membrane depolarization (PubMed:22412352). Unlike theCC       single srkA disruption mutant, a triple srkA-mazE-mazF disruptionCC       mutant shows no hyperlethality in the presence of nalidixic acidCC       or UV light, suggesting SrkA has a negative effect on MazFCC       (PubMed:23416055). mRNA interferases play a role in bacterialCC       persistence to antibiotics; as 10 mRNA interferases areCC       successively deleted reduced levels of persisters are generatedCC       (PubMed:21788497). {ECO:0000269|PubMed:11222603,CC       ECO:0000269|PubMed:12972253, ECO:0000269|PubMed:15150257,CC       ECO:0000269|PubMed:15316771, ECO:0000269|PubMed:19707553,CC       ECO:0000269|PubMed:20005847, ECO:0000269|PubMed:21788497,CC       ECO:0000269|PubMed:22412352, ECO:0000269|PubMed:23416055,CC       ECO:0000269|PubMed:24375411, ECO:0000269|PubMed:8650219}.CC   \\n']"
      ]
     },
     "execution_count": 60,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#the raw file contains too many unnecessary lines. Remove all line-breaker, and add line-breaker only ahead of the start of a gene(containing ID) and localization info:\n",
    "\n",
    "f = open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/uniprot-k12-filtered-organism%3A_Escherichia+coli+%28strain+K12%29+%5B83333%5D--.txt', 'r')\n",
    "\n",
    "f_str= f.read()\n",
    "# Check out the first 10 entries\n",
    "#f_list=f.readlines()\n",
    "f_str=f_str.replace('\\n','')\n",
    "f_str=f_str.replace('ID   ','\\n ID   ')\n",
    "f_str=f_str.replace('-!-','\\n -!-')\n",
    "\n",
    "#save the parsed file\n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/K12_parsed.txt', 'w') as output:\n",
    "    output.write(f_str)\n",
    "parsed = open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/K12_parsed.txt', 'r')\n",
    "par_list= parsed.readlines()\n",
    "#par_list[:100]\n",
    "#for line in parsed:\n",
    "    #if line[:6]==' ID   '\n",
    "par_list[:10]\n",
    "\n"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[' ID   MAZF_ECOLI   \\n',\n",
       " ' ID   HNS_ECOLI    \\n',\n",
       " \" SUBCELLULAR LOCATION: Cytoplasm, nucleoidCC       {ECO:0000269|PubMed:21903814, ECO:0000269|PubMed:333393}.CC       Note=Forms 2 compact clusters per chromosome, located at one-CC       quarter and three-quarter positions on the cell's long axis: 2 H-CC       NS-repressed genes were closely associated with H-NS clustersCC       while a non-H-NS-repressed gene was not substantially associatedCC       with the H-NS cluster (PubMed:21903814).CC       {ECO:0000269|PubMed:21903814}.CC   \\n\",\n",
       " ' ID   ERA_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:8282709}. CellCC       inner membrane {ECO:0000269|PubMed:8282709}; Peripheral membraneCC       protein {ECO:0000269|PubMed:8282709}; Cytoplasmic sideCC       {ECO:0000269|PubMed:8282709}. Note=Binding is GDP or GTP-CC       dependent, slightly more protein is bound in the presence of GTPCC       than GDP.CC   \\n',\n",
       " ' ID   RSEA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:11777003, ECO:0000269|PubMed:12183369,CC       ECO:0000269|PubMed:9159522, ECO:0000269|PubMed:9159523}; Single-CC       pass type II membrane protein {ECO:0000269|PubMed:11777003,CC       ECO:0000269|PubMed:12183369, ECO:0000269|PubMed:9159522,CC       ECO:0000269|PubMed:9159523}. Note=Following cleavage by DegS theCC       large fragment of the protein is still in the inner membrane andCC       retains its anti-sigma-E activity.CC   \\n',\n",
       " ' ID   CSRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00167}.CC   \\n',\n",
       " ' ID   PHOQ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   MAZE_ECOLI   \\n',\n",
       " ' ID   OBG_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01454,CC       ECO:0000269|PubMed:15292126}. Note=The majority is associated withCC       the 50S ribosomal subunit (PubMed:15292126), another group alsoCC       sees association with 30S subunit (PubMed:15836769).CC       {ECO:0000269|PubMed:15292126, ECO:0000269|PubMed:15836769}.CC   \\n',\n",
       " ' ID   CPXA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:25207645,CC       ECO:0000269|PubMed:3058985}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:15919996, ECO:0000305|PubMed:3058985}.CC   \\n',\n",
       " ' ID   SULA_ECOLI   \\n',\n",
       " ' ID   RPOE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:11777003,CC       ECO:0000269|PubMed:9159522}. Note=Associates with the innerCC       membrane via RseA (PubMed:9159522, PubMed:11777003).CC       {ECO:0000269|PubMed:11777003}.CC   \\n',\n",
       " ' ID   RSSB_ECOLI   \\n',\n",
       " ' ID   MQSR_ECOLI   \\n',\n",
       " ' ID   HIPA_ECOLI   \\n',\n",
       " ' ID   FTSZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00909,CC       ECO:0000269|PubMed:1944597, ECO:0000269|PubMed:8917533}.CC       Note=Assembles at midcell at the inner surface of the cytoplasmicCC       membrane.CC   \\n',\n",
       " ' ID   FTSH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01458, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:8444797, ECO:0000269|PubMed:8947034}; Multi-CC       pass membrane protein {ECO:0000255|HAMAP-Rule:MF_01458,CC       ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:8444797,CC       ECO:0000269|PubMed:8947034}.CC   \\n',\n",
       " ' ID   BAMA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_01430, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:23882017, ECO:0000269|PubMed:24619089,CC       ECO:0000269|PubMed:24914988, ECO:0000269|PubMed:26901871,CC       ECO:0000269|PubMed:9298646}.CC   \\n',\n",
       " ' ID   RNR_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   RELE_ECOLI   \\n',\n",
       " ' ID   RSGA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01820,CC       ECO:0000269|PubMed:14973029}. Note=Associates with 30S ribosomesCC       (PubMed:14973029). {ECO:0000269|PubMed:14973029}.CC   \\n',\n",
       " ' ID   RSEP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:11689431, ECO:0000269|PubMed:11750129,CC       ECO:0000269|PubMed:11867724, ECO:0000269|PubMed:15919996}; Multi-CC       pass membrane protein {ECO:0000269|PubMed:11689431,CC       ECO:0000269|PubMed:11750129, ECO:0000269|PubMed:11867724,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   NAGA_ECOLI   \\n',\n",
       " ' ID   AHPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:7499381}.CC   \\n',\n",
       " ' ID   IMDH_ECOLI   \\n',\n",
       " ' ID   DCEB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:12912902}.CC       Membrane {ECO:0000269|PubMed:12912902}. Note=Localized exclusivelyCC       in the cytoplasm at neutral pH, but is recruited to the membraneCC       when the pH falls.CC   \\n',\n",
       " ' ID   DPS_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm, nucleoid.CC   \\n',\n",
       " ' ID   AAEX_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000255|HAMAP-CC       Rule:MF_01546}; Multi-pass membrane protein {ECO:0000255|HAMAP-CC       Rule:MF_01546}.CC   \\n',\n",
       " ' ID   ACCD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   ACDH_ECOLI   \\n',\n",
       " ' ID   ACPS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   ACPT_ECOLI   \\n',\n",
       " ' ID   ACP_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   ACRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15228545, ECO:0000269|PubMed:16079137}; Lipid-CC       anchor {ECO:0000255|PROSITE-ProRule:PRU00303,CC       ECO:0000269|PubMed:15228545, ECO:0000269|PubMed:16079137}. Note=AnCC       unlipidated version of this protein (directed to the periplasm byCC       the OmpA signal sequence) functions normally.CC   \\n',\n",
       " ' ID   ACYP_ECOLI   \\n',\n",
       " ' ID   ADA_ECOLI    \\n',\n",
       " ' ID   AFUB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane protein.CC       Note=When overexpressed using vectors that provide a promoter andCC       ribosome binding site (PubMed:15919996).CC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   AGAI_ECOLI   \\n',\n",
       " ' ID   AGAR_ECOLI   \\n',\n",
       " ' ID   ARAB_ECOLI   \\n',\n",
       " ' ID   ARAH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ARGK_ECOLI   \\n',\n",
       " ' ID   ARNA_ECOLI   \\n',\n",
       " ' ID   ARNE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ASCG_ECOLI   \\n',\n",
       " ' ID   ASNA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   ASPG2_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   ASTA_ECOLI   \\n',\n",
       " ' ID   ARTA_ECOLI   \\n',\n",
       " ' ID   ARTI_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:8801422}.CC   \\n',\n",
       " ' ID   ATDA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   ATPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01346, ECO:0000269|PubMed:16079137}; Peripheral membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01346,CC       ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   ATPF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01398}; Single-pass membrane protein {ECO:0000255|HAMAP-CC       Rule:MF_01398}.CC   \\n',\n",
       " ' ID   AVTA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250}.CC   \\n',\n",
       " ' ID   AROP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ARPA_ECOLI   \\n',\n",
       " ' ID   ASCB_ECOLI   \\n',\n",
       " ' ID   ASNC_ECOLI   \\n',\n",
       " ' ID   ATL_ECOLI    \\n',\n",
       " ' ID   ATMA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ATOS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:16153782}; Multi-CC       pass membrane protein {ECO:0000255}.CC   \\n',\n",
       " ' ID   ASTB_ECOLI   \\n',\n",
       " ' ID   ATPB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Peripheral membraneCC       protein.CC   \\n',\n",
       " ' ID   BETB_ECOLI   \\n',\n",
       " ' ID   BETI_ECOLI   \\n',\n",
       " ' ID   BFR_ECOLI    \\n',\n",
       " ' ID   ARTM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ASTD_ECOLI   \\n',\n",
       " ' ID   ATOA_ECOLI   \\n',\n",
       " ' ID   ATOB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   ATOD_ECOLI   \\n',\n",
       " ' ID   ASLB_ECOLI   \\n',\n",
       " ' ID   ASTC_ECOLI   \\n',\n",
       " ' ID   ATOE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; U00096; AAC75283.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA16019.1; -; Genomic_DNA.DR   PIR; E64992; E64992.DR   RefSeq; NP_416727.1; NC_000913.3.DR   RefSeq; WP_000580275.1; NZ_LN832404.1.DR   ProteinModelPortal; P76460; -.DR   BioGrid; 4261814; 7.DR   STRING; 316385.ECDH10B_2381; -.DR   TCDB; 2.A.73.1.1; the short chain fatty acid uptake (atoe) family.DR   PaxDb; P76460; -.DR   PRIDE; P76460; -.DR   EnsemblBacteria; AAC75283; AAC75283; b2223.DR   EnsemblBacteria; BAA16019; BAA16019; BAA16019.DR   GeneID; 946721; -.DR   KEGG; ecj:JW2217; -.DR   KEGG; eco:b2223; -.DR   PATRIC; fig|1411691.4.peg.12; -.DR   EchoBASE; EB1622; -.DR   EcoGene; EG11671; atoE.DR   eggNOG; ENOG4105CRW; Bacteria.DR   eggNOG; COG2031; LUCA.DR   HOGENOM; HOG000295152; -.DR   InParanoid; P76460; -.DR   KO; K02106; -.DR   OMA; QDKTIAH; -.DR   PhylomeDB; P76460; -.DR   BioCyc; EcoCyc:EG11671-MONOMER; -.DR   BioCyc; MetaCyc:EG11671-MONOMER; -.DR   PRO; PR:P76460; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0016021; C:integral component of membrane; IEA:UniProtKB-KW.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   InterPro; IPR006161; CHP00366.DR   InterPro; IPR006160; SCFA_transpt_AtoE.DR   PANTHER; PTHR41983; PTHR41983; 1.DR   Pfam; PF02667; SCFA_trans; 1.DR   TIGRFAMs; TIGR00366; TIGR00366; 1.PE   1: Evidence at protein level;KW   Cell inner membrane; Cell membrane; Complete proteome; Membrane;KW   Reference proteome; Transmembrane; Transmembrane helix; Transport.FT   CHAIN         1    440       Putative short-chain fatty acidFT                                transporter.FT                                /FTId=PRO_0000064733.FT   TOPO_DOM      1     19       Periplasmic. {ECO:0000255}.FT   TRANSMEM     20     40       Helical. {ECO:0000255}.FT   TOPO_DOM     41     53       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM     54     74       Helical. {ECO:0000255}.FT   TOPO_DOM     75    102       Periplasmic. {ECO:0000255}.FT   TRANSMEM    103    123       Helical. {ECO:0000255}.FT   TOPO_DOM    124    137       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    138    158       Helical. {ECO:0000255}.FT   TRANSMEM    159    179       Helical. {ECO:0000255}.FT   TOPO_DOM    180    180       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    181    201       Helical. {ECO:0000255}.FT   TOPO_DOM    202    244       Periplasmic. {ECO:0000255}.FT   TRANSMEM    245    265       Helical. {ECO:0000255}.FT   TRANSMEM    266    286       Helical. {ECO:0000255}.FT   TOPO_DOM    287    313       Periplasmic. {ECO:0000255}.FT   TRANSMEM    314    334       Helical. {ECO:0000255}.FT   TOPO_DOM    335    351       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    352    372       Helical. {ECO:0000255}.FT   TOPO_DOM    373    394       Periplasmic. {ECO:0000255}.FT   TRANSMEM    395    415       Helical. {ECO:0000255}.FT   TOPO_DOM    416    419       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    420    440       Helical. {ECO:0000255}.SQ   SEQUENCE   440 AA;  47528 MW;  5C9D5ECE79F5FA71 CRC64;     MIGRISRFMT RFVSRWLPDP LIFAMLLTLL TFVIALWLTP QTPISMVKMW GDGFWNLLAF     GMQMALIIVT GHALASSAPV KSLLRTAASA AKTPVQGVML VTFFGSVACV INWGFGLVVG     AMFAREVARR VPGSDYPLLI ACAYIGFLTW GGGFSGSMPL LAATPGNPVE HIAGLIPVGD     TLFSGFNIFI TVALIVVMPF ITRMMMPKPS DVVSIDPKLL MEEADFQKQL PKDAPPSERL     EESRILTLII GALGIAYLAM YFSEHGFNIT INTVNLMFMI AGLLLHKTPM AYMRAISAAA     RSTAGILVQF PFYAGIQLMM EHSGLGGLIT EFFINVANKD TFPVMTFFSS ALINFAVPSG     GGHWVIQGPF VIPAAQALGA DLGKSVMAIA YGEQWMNMAQ PFWALPALAI AGLGVRDIMG     YCITALLFSG VIFVIGLTLF//\\n',\n",
       " ' ID   BAES_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ATP6_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01393, ECO:0000269|PubMed:15919996}; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01393,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ATPD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01416, ECO:0000269|PubMed:16079137}; Peripheral membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01416,CC       ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   ATPZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   AZOR_ECOLI   \\n',\n",
       " ' ID   BAER_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   BDCR_ECOLI   \\n',\n",
       " ' ID   BETA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000269|PubMed:3512525};CC       Peripheral membrane protein {ECO:0000269|PubMed:3512525}.CC   \\n',\n",
       " ' ID   BGLA_ECOLI   \\n',\n",
       " ' ID   BGLX_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   ATPE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:21778229};CC       Peripheral membrane protein {ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:21778229}.CC   \\n',\n",
       " ' ID   BAMC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00924, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00924,CC       ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BDCA_ECOLI   \\n',\n",
       " ' ID   BEPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000255|HAMAP-Rule:MF_00997,CC       ECO:0000269|PubMed:22491786, ECO:0000269|PubMed:24003122}. Note=ACC       significant amount of BepA is membrane-associated. ThisCC       localization could result from interaction with the BAM complex.CC   \\n',\n",
       " ' ID   BGLB_ECOLI   \\n',\n",
       " ' ID   BASR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   BCSB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Single-pass type ICC       membrane protein.CC   \\n',\n",
       " ' ID   BCSC_ECOLI   \\n',\n",
       " ' ID   BCSG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Multi-CC       pass membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   BGA2_ECOLI   \\n',\n",
       " ' ID   BHSA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:22089859}.CC   \\n',\n",
       " ' ID   BIOD2_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00336}.CC   \\n',\n",
       " ' ID   BIOP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000269|Ref.5};CC       Multi-pass membrane protein {ECO:0000255}.CC   \\n',\n",
       " ' ID   BORD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000305}; Lipid-anchorCC       {ECO:0000305}.CC   \\n',\n",
       " ' ID   AZUC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000305|PubMed:19121005, ECO:0000305|PubMed:21778229};CC       Peripheral membrane protein {ECO:0000305|PubMed:19121005,CC       ECO:0000305|PubMed:21778229}.CC   \\n',\n",
       " ' ID   BAMB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00923, ECO:0000269|PubMed:15851030,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00923,CC       ECO:0000269|PubMed:15851030, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BAMD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00922, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00922,CC       ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BAX_ECOLI    \\n',\n",
       " ' ID   BIOD1_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   BLC_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Lipid-anchor.CC   \\n',\n",
       " ' ID   BLUF_ECOLI   \\n',\n",
       " ' ID   BTSR_ECOLI   \\n',\n",
       " ' ID   BTSS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000255}.CC   \\n',\n",
       " ' ID   ATPG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:16079137}; Peripheral membrane proteinCC       {ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   ATPL_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BAME_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00925, ECO:0000269|PubMed:17404237,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00925,CC       ECO:0000269|PubMed:17404237, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BASS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   BCCP_ECOLI   \\n',\n",
       " ' ID   BDM_ECOLI    \\n',\n",
       " ' ID   BGLG_ECOLI   \\n',\n",
       " ' ID   BICB_ECOLI   \\n',\n",
       " ' ID   BIOC_ECOLI   \\n',\n",
       " ' ID   BIRA_ECOLI   \\n',\n",
       " ' ID   BARA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BCR_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BCSQP_ECOLI  \\n',\n",
       " ' ID   BGAL_ECOLI   \\n',\n",
       " ' ID   BGLH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000305|PubMed:10027967}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:10027967}. Note=Can be easily removed from theCC       outer membrane by treatment with detergent.CC   \\n',\n",
       " ' ID   BGLJ_ECOLI   \\n',\n",
       " ' ID   BCP_ECOLI    \\n',\n",
       " ' ID   BCSA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Multi-CC       pass membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   BCSE_ECOLI   \\n',\n",
       " ' ID   BCSF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Single-CC       pass membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   BCSQ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:19400787}.CC       Note=Localizes at the cell pole, this does not require any genesCC       of the divergently expressed yhjRbcsQABZC and bcsEFG operons.CC       {ECO:0000269|PubMed:19400787}.CC   \\n',\n",
       " ' ID   BEEE_ECOLI   \\n',\n",
       " ' ID   BETT_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BFD_ECOLI    \\n',\n",
       " ' ID   BGLR_ECOLI   \\n',\n",
       " ' ID   BIOH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250}.CC   \\n',\n",
       " ' ID   BOLA_ECOLI   \\n',\n",
       " ' ID   BSSR_ECOLI   \\n',\n",
       " ' ID   BTST_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:22685278,CC       ECO:0000269|PubMed:29061664}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:29061664}.CC   \\n',\n",
       " ' ID   MENF_ECOLI   \\n',\n",
       " ' ID   MENH_ECOLI   \\n',\n",
       " ' ID   MEPM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Single-CC       pass membrane protein {ECO:0000305}. Note=Uniform peripheralCC       location, with partial enrichment at cell poles.CC   \\n',\n",
       " ' ID   METAS_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00295,CC       ECO:0000305}.CC   \\n',\n",
       " ' ID   METJ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MIOC_ECOLI   \\n',\n",
       " ' ID   MLAF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000305|PubMed:19383799}; Peripheral membrane proteinCC       {ECO:0000305|PubMed:19383799}; Cytoplasmic sideCC       {ECO:0000305|PubMed:19383799}.CC   \\n',\n",
       " ' ID   MMUM_ECOLI   \\n',\n",
       " ' ID   MNGR_ECOLI   \\n',\n",
       " ' ID   MNMG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:17062623}.CC   \\n',\n",
       " ' ID   MELR_ECOLI   \\n',\n",
       " ' ID   MEND_ECOLI   \\n',\n",
       " ' ID   MEPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305|PubMed:15292190}.CC   \\n',\n",
       " ' ID   METK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00086}.CC   \\n',\n",
       " ' ID   MLAB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305|PubMed:19383799}.CC   \\n',\n",
       " ' ID   MLAC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305|PubMed:19383799}.CC   \\n',\n",
       " ' ID   MNMA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MOBA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   KDGT_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   KDPB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00285, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2849541}; Multi-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_00285, ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   KDSC_ECOLI   \\n',\n",
       " ' ID   KDSD_ECOLI   \\n',\n",
       " ' ID   KDTA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000305|PubMed:1577828}; Single-pass membrane proteinCC       {ECO:0000305|PubMed:1577828}; Cytoplasmic sideCC       {ECO:0000305|PubMed:1577828}.CC   \\n',\n",
       " ' ID   KDUD_ECOLI   \\n',\n",
       " ' ID   KEFB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01412, ECO:0000269|PubMed:15919996}; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01412,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   KEFC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01413, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2046548}; Multi-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_01413, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2046548}.CC   \\n',\n",
       " ' ID   KDPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00275, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2849541, ECO:0000269|PubMed:7896809}; Multi-CC       pass membrane protein {ECO:0000255|HAMAP-Rule:MF_00275,CC       ECO:0000269|PubMed:7896809}.CC   \\n',\n",
       " ' ID   KDPE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:1532388}.CC   \\n',\n",
       " ' ID   KGUA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   KILR_ECOLI   \\n',\n",
       " ' ID   KITH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00124}.CC   \\n',\n",
       " ' ID   KPYK1_ECOLI  \\n',\n",
       " ' ID   KPYK2_ECOLI  \\n',\n",
       " ' ID   LACI_ECOLI   \\n',\n",
       " ' ID   LPMG_ECOLI   \\n',\n",
       " ' ID   LPTB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:16079137}.CC       Cell inner membrane {ECO:0000269|PubMed:16079137}; PeripheralCC       membrane protein {ECO:0000269|PubMed:16079137}; Cytoplasmic sideCC       {ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   LSRG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_02051}.CC   \\n',\n",
       " ' ID   LSRR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   LTND_ECOLI   \\n',\n",
       " ' ID   LYSR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; J01614; AAA83862.1; -; Genomic_DNA.DR   EMBL; U29581; AAB40486.1; -; Genomic_DNA.DR   EMBL; U00096; AAC75878.1; -; Genomic_DNA.DR   EMBL; AP009048; BAE76908.1; -; Genomic_DNA.DR   EMBL; J03732; AAA23471.1; -; Genomic_DNA.DR   PIR; A03565; RGECK.DR   RefSeq; NP_417316.1; NC_000913.3.DR   RefSeq; WP_000741825.1; NZ_LN832404.1.DR   ProteinModelPortal; P03030; -.DR   SMR; P03030; -.DR   BioGrid; 4262309; 10.DR   DIP; DIP-10135N; -.DR   IntAct; P03030; 6.DR   STRING; 316385.ECDH10B_3011; -.DR   PaxDb; P03030; -.DR   PRIDE; P03030; -.DR   EnsemblBacteria; AAC75878; AAC75878; b2839.DR   EnsemblBacteria; BAE76908; BAE76908; BAE76908.DR   GeneID; 947311; -.DR   KEGG; ecj:JW2807; -.DR   KEGG; eco:b2839; -.DR   PATRIC; fig|1411691.4.peg.3895; -.DR   EchoBASE; EB0546; -.DR   EcoGene; EG10551; lysR.DR   eggNOG; ENOG4105RQF; Bacteria.DR   eggNOG; ENOG410XT7K; LUCA.DR   HOGENOM; HOG000233527; -.DR   InParanoid; P03030; -.DR   OMA; FHAVMTT; -.DR   PhylomeDB; P03030; -.DR   BioCyc; EcoCyc:PD00360; -.DR   PRO; PR:P03030; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0005737; C:cytoplasm; IEA:UniProtKB-SubCell.DR   GO; GO:0003677; F:DNA binding; IEA:UniProtKB-KW.DR   GO; GO:0003700; F:DNA binding transcription factor activity; IEA:InterPro.DR   GO; GO:0009089; P:lysine biosynthetic process via diaminopimelate; IDA:EcoCyc.DR   GO; GO:0010628; P:positive regulation of gene expression; IMP:EcoCyc.DR   GO; GO:0006351; P:transcription, DNA-templated; IEA:UniProtKB-KW.DR   CDD; cd08456; PBP2_LysR; 1.DR   Gene3D; 1.10.10.10; -; 1.DR   InterPro; IPR037414; LysR_PBP2.DR   InterPro; IPR005119; LysR_subst-bd.DR   InterPro; IPR000847; Tscrpt_reg_HTH_LysR.DR   InterPro; IPR036388; WH-like_DNA-bd_sf.DR   InterPro; IPR036390; WH_DNA-bd_sf.DR   Pfam; PF00126; HTH_1; 1.DR   Pfam; PF03466; LysR_substrate; 1.DR   PRINTS; PR00039; HTHLYSR.DR   SUPFAM; SSF46785; SSF46785; 1.DR   PROSITE; PS50931; HTH_LYSR; 1.PE   4: Predicted;KW   Activator; Complete proteome; Cytoplasm; DNA-binding;KW   Reference proteome; Repressor; Transcription;KW   Transcription regulation.FT   CHAIN         1    311       Transcriptional activator protein LysR.FT                                /FTId=PRO_0000105669.FT   DOMAIN        1     61       HTH lysR-type. {ECO:0000255|PROSITE-FT                                ProRule:PRU00253}.FT   DNA_BIND     21     40       H-T-H motif. {ECO:0000255|PROSITE-FT                                ProRule:PRU00253}.SQ   SEQUENCE   311 AA;  34365 MW;  3E1913BB6A2CF8F9 CRC64;     MAAVNLRHIE IFHAVMTAGS LTEAAHLLHT SQPTVSRELA RFEKVIGLKL FERVRGRLHP     TVQGLRLFEE VQRSWYGLDR IVSAAESLRE FRQGELSIAC LPVFSQSFLP QLLQPFLARY     PDVSLNIVPQ ESPLLEEWLS AQRHDLGLTE TLHTPAGTER TELLSLDEVC VLPPGHPLAV     KKVLTPDDFQ GENYISLSRT DSYRQLLDQL FTEHQVKRRM IVETHSAASV CAMVRAGVGI     SVVNPLTALD YAASGLVVRR FSIAVPFTVS LIRPLHRPSS ALVQAFSGHL QAGLPKLVTS     LDAILSSATT A//\\n',\n",
       " ' ID   MARR_ECOLI   \\n',\n",
       " ' ID   MASZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MIAB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01864}.CC   \\n',\n",
       " ' ID   MNGA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|PROSITE-CC       ProRule:PRU00427, ECO:0000305|PubMed:14645248}; Multi-passCC       membrane protein {ECO:0000255|PROSITE-ProRule:PRU00427,CC       ECO:0000305|PubMed:14645248}.CC   \\n',\n",
       " ' ID   MNMC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   MNTH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00221, ECO:0000269|PubMed:14607838}; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_00221,CC       ECO:0000269|PubMed:14607838}.CC   \\n',\n",
       " ' ID   MPL_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Secreted.CC   \\n',\n",
       " ' ID   KCY_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   KDPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00276, ECO:0000269|PubMed:2849541}; Single-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_00276}.CC   \\n',\n",
       " ' ID   KDSB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00057}.CC   \\n',\n",
       " ' ID   MLIC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:18369469}; Lipid-anchor {ECO:0000255|PROSITE-CC       ProRule:PRU00303, ECO:0000269|PubMed:18369469}. Note=Anchored toCC       the periplasmic side. {ECO:0000269|PubMed:18369469}.CC   \\n',\n",
       " ' ID   MLTA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Lipid-anchor.CC   \\n',\n",
       " ' ID   MLTB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Lipid-anchor;CC       Periplasmic side.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; U18785; AAB60060.1; -; Genomic_DNA.DR   EMBL; U00096; AAC75743.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA16563.1; -; Genomic_DNA.DR   EMBL; J02708; -; NOT_ANNOTATED_CDS; Genomic_DNA.DR   PIR; A65050; A65050.DR   RefSeq; NP_417181.1; NC_000913.3.DR   RefSeq; WP_001295177.1; NZ_LN832404.1.DR   PDB; 1D0K; X-ray; 2.02 A; A=40-361.DR   PDB; 1D0L; X-ray; 1.97 A; A=40-361.DR   PDB; 1D0M; X-ray; 2.47 A; A=40-361.DR   PDB; 1LTM; X-ray; 1.70 A; A=42-361.DR   PDB; 1QDR; X-ray; 2.10 A; A=40-361.DR   PDB; 1QDT; X-ray; 2.10 A; A=40-361.DR   PDB; 1QUS; X-ray; 1.70 A; A=40-361.DR   PDB; 1QUT; X-ray; 2.44 A; A=40-361.DR   PDBsum; 1D0K; -.DR   PDBsum; 1D0L; -.DR   PDBsum; 1D0M; -.DR   PDBsum; 1LTM; -.DR   PDBsum; 1QDR; -.DR   PDBsum; 1QDT; -.DR   PDBsum; 1QUS; -.DR   PDBsum; 1QUT; -.DR   ProteinModelPortal; P41052; -.DR   SMR; P41052; -.DR   BioGrid; 4262079; 390.DR   DIP; DIP-10221N; -.DR   IntAct; P41052; 10.DR   STRING; 316385.ECDH10B_2869; -.DR   BindingDB; P41052; -.DR   ChEMBL; CHEMBL2046262; -.DR   DrugBank; DB03709; Bicine.DR   DrugBank; DB02595; Bulgecin A.DR   CAZy; GH103; Glycoside Hydrolase Family 103.DR   PaxDb; P41052; -.DR   PRIDE; P41052; -.DR   EnsemblBacteria; AAC75743; AAC75743; b2701.DR   EnsemblBacteria; BAA16563; BAA16563; BAA16563.DR   GeneID; 947184; -.DR   KEGG; ecj:JW2671; -.DR   KEGG; eco:b2701; -.DR   PATRIC; fig|511145.12.peg.2792; -.DR   EchoBASE; EB2561; -.DR   EcoGene; EG12699; mltB.DR   eggNOG; ENOG4105CVE; Bacteria.DR   eggNOG; COG2951; LUCA.DR   HOGENOM; HOG000257828; -.DR   InParanoid; P41052; -.DR   KO; K08305; -.DR   OMA; SSIRDYA; -.DR   PhylomeDB; P41052; -.DR   BioCyc; EcoCyc:G7410-MONOMER; -.DR   BioCyc; MetaCyc:G7410-MONOMER; -.DR   EvolutionaryTrace; P41052; -.DR   PRO; PR:P41052; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0009279; C:cell outer membrane; IDA:EcoCyc.DR   GO; GO:0030288; C:outer membrane-bounded periplasmic space; TAS:EcoCyc.DR   GO; GO:0005509; F:calcium ion binding; IDA:EcoCyc.DR   GO; GO:0016829; F:lyase activity; IEA:UniProtKB-KW.DR   GO; GO:0008932; F:lytic endotransglycosylase activity; IDA:EcoCyc.DR   GO; GO:0008933; F:lytic transglycosylase activity; IDA:EcoCyc.DR   GO; GO:0031402; F:sodium ion binding; IDA:EcoCyc.DR   GO; GO:0071555; P:cell wall organization; IEA:UniProtKB-KW.DR   GO; GO:0009253; P:peptidoglycan catabolic process; IDA:EcoCyc.DR   InterPro; IPR023346; Lysozyme-like_dom_sf.DR   InterPro; IPR011757; Lytic_transglycosylase_MltB.DR   InterPro; IPR031304; SLT_2.DR   PANTHER; PTHR30163:SF0; PTHR30163:SF0; 1.DR   Pfam; PF13406; SLT_2; 1.DR   SUPFAM; SSF53955; SSF53955; 1.DR   TIGRFAMs; TIGR02282; MltB; 1.DR   PROSITE; PS51257; PROKAR_LIPOPROTEIN; 1.PE   1: Evidence at protein level;KW   3D-structure; Cell outer membrane; Cell wall biogenesis/degradation;KW   Complete proteome; Lipoprotein; Lyase; Membrane; Palmitate;KW   Reference proteome; Signal.FT   SIGNAL        1     18       {ECO:0000305}.FT   CHAIN        19    361       Membrane-bound lytic mureinFT                                transglycosylase B.FT                                /FTId=PRO_0000032784.FT   ACT_SITE    162    162FT   LIP\\n',\n",
       " ' ID   MLTD_ECOLI   \\n']"
      ]
     },
     "execution_count": 77,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#From parsed file, slice lines starting with ID and subcellular location:\n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/K12_parsed.txt', 'r') as f, open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/K12_IDlocal.txt', 'w') as f_out:\n",
    "    f.seek(0)\n",
    "    f_list = f.readlines()\n",
    "    for i,line in enumerate(f_list):\n",
    "        if f_list[i][:6]==' ID   ' and 'ECOLI' in line[:34]:\n",
    "            f_out.write(line[:19])\n",
    "            f_out.write('\\n')\n",
    "        if line[:25]==' -!- SUBCELLULAR LOCATION':\n",
    "            f_out.write(line[4:])\n",
    "        \n",
    "            \n",
    "            \n",
    "\n",
    "x = open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/K12_IDlocal.txt', 'r')\n",
    "x_list = x.readlines()\n",
    "x_list[:300]\n",
    "            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "6609"
      ]
     },
     "execution_count": 72,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#check number of items in a list:\n",
    "size = len(x_list)\n",
    "size"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['\\n',\n",
       " ' ID   MAZF_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   HNS_ECOLI    \\n',\n",
       " \" SUBCELLULAR LOCATION: Cytoplasm, nucleoidCC       {ECO:0000269|PubMed:21903814, ECO:0000269|PubMed:333393}.CC       Note=Forms 2 compact clusters per chromosome, located at one-CC       quarter and three-quarter positions on the cell's long axis: 2 H-CC       NS-repressed genes were closely associated with H-NS clustersCC       while a non-H-NS-repressed gene was not substantially associatedCC       with the H-NS cluster (PubMed:21903814).CC       {ECO:0000269|PubMed:21903814}.CC   \\n\",\n",
       " ' ID   ERA_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:8282709}. CellCC       inner membrane {ECO:0000269|PubMed:8282709}; Peripheral membraneCC       protein {ECO:0000269|PubMed:8282709}; Cytoplasmic sideCC       {ECO:0000269|PubMed:8282709}. Note=Binding is GDP or GTP-CC       dependent, slightly more protein is bound in the presence of GTPCC       than GDP.CC   \\n',\n",
       " ' ID   RSEA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:11777003, ECO:0000269|PubMed:12183369,CC       ECO:0000269|PubMed:9159522, ECO:0000269|PubMed:9159523}; Single-CC       pass type II membrane protein {ECO:0000269|PubMed:11777003,CC       ECO:0000269|PubMed:12183369, ECO:0000269|PubMed:9159522,CC       ECO:0000269|PubMed:9159523}. Note=Following cleavage by DegS theCC       large fragment of the protein is still in the inner membrane andCC       retains its anti-sigma-E activity.CC   \\n',\n",
       " ' ID   CSRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00167}.CC   \\n',\n",
       " ' ID   PHOQ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   MAZE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   OBG_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01454,CC       ECO:0000269|PubMed:15292126}. Note=The majority is associated withCC       the 50S ribosomal subunit (PubMed:15292126), another group alsoCC       sees association with 30S subunit (PubMed:15836769).CC       {ECO:0000269|PubMed:15292126, ECO:0000269|PubMed:15836769}.CC   \\n',\n",
       " ' ID   CPXA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:25207645,CC       ECO:0000269|PubMed:3058985}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:15919996, ECO:0000305|PubMed:3058985}.CC   \\n',\n",
       " ' ID   SULA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   RPOE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:11777003,CC       ECO:0000269|PubMed:9159522}. Note=Associates with the innerCC       membrane via RseA (PubMed:9159522, PubMed:11777003).CC       {ECO:0000269|PubMed:11777003}.CC   \\n',\n",
       " ' ID   RSSB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MQSR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   HIPA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   FTSZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00909,CC       ECO:0000269|PubMed:1944597, ECO:0000269|PubMed:8917533}.CC       Note=Assembles at midcell at the inner surface of the cytoplasmicCC       membrane.CC   \\n',\n",
       " ' ID   FTSH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01458, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:8444797, ECO:0000269|PubMed:8947034}; Multi-CC       pass membrane protein {ECO:0000255|HAMAP-Rule:MF_01458,CC       ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:8444797,CC       ECO:0000269|PubMed:8947034}.CC   \\n',\n",
       " ' ID   BAMA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_01430, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:23882017, ECO:0000269|PubMed:24619089,CC       ECO:0000269|PubMed:24914988, ECO:0000269|PubMed:26901871,CC       ECO:0000269|PubMed:9298646}.CC   \\n',\n",
       " ' ID   RNR_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   RELE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   RSGA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01820,CC       ECO:0000269|PubMed:14973029}. Note=Associates with 30S ribosomesCC       (PubMed:14973029). {ECO:0000269|PubMed:14973029}.CC   \\n',\n",
       " ' ID   RSEP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:11689431, ECO:0000269|PubMed:11750129,CC       ECO:0000269|PubMed:11867724, ECO:0000269|PubMed:15919996}; Multi-CC       pass membrane protein {ECO:0000269|PubMed:11689431,CC       ECO:0000269|PubMed:11750129, ECO:0000269|PubMed:11867724,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   NAGA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   AHPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:7499381}.CC   \\n',\n",
       " ' ID   IMDH_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   DCEB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:12912902}.CC       Membrane {ECO:0000269|PubMed:12912902}. Note=Localized exclusivelyCC       in the cytoplasm at neutral pH, but is recruited to the membraneCC       when the pH falls.CC   \\n',\n",
       " ' ID   DPS_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm, nucleoid.CC   \\n',\n",
       " ' ID   AAEX_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000255|HAMAP-CC       Rule:MF_01546}; Multi-pass membrane protein {ECO:0000255|HAMAP-CC       Rule:MF_01546}.CC   \\n',\n",
       " ' ID   ACCD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   ACDH_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ACPS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   ACPT_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ACP_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   ACRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15228545, ECO:0000269|PubMed:16079137}; Lipid-CC       anchor {ECO:0000255|PROSITE-ProRule:PRU00303,CC       ECO:0000269|PubMed:15228545, ECO:0000269|PubMed:16079137}. Note=AnCC       unlipidated version of this protein (directed to the periplasm byCC       the OmpA signal sequence) functions normally.CC   \\n',\n",
       " ' ID   ACYP_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ADA_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   AFUB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane protein.CC       Note=When overexpressed using vectors that provide a promoter andCC       ribosome binding site (PubMed:15919996).CC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   AGAI_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   AGAR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ARAB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ARAH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ARGK_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ARNA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ARNE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ASCG_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ASNA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   ASPG2_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   ASTA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ARTA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ARTI_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:8801422}.CC   \\n',\n",
       " ' ID   ATDA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   ATPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01346, ECO:0000269|PubMed:16079137}; Peripheral membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01346,CC       ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   ATPF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01398}; Single-pass membrane protein {ECO:0000255|HAMAP-CC       Rule:MF_01398}.CC   \\n',\n",
       " ' ID   AVTA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250}.CC   \\n',\n",
       " ' ID   AROP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ARPA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ASCB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ASNC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ATL_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   ATMA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ATOS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:16153782}; Multi-CC       pass membrane protein {ECO:0000255}.CC   \\n',\n",
       " ' ID   ASTB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ATPB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Peripheral membraneCC       protein.CC   \\n',\n",
       " ' ID   BETB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BETI_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BFR_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   ARTM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ASTD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ATOA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ATOB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   ATOD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ASLB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ASTC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ATOE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; U00096; AAC75283.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA16019.1; -; Genomic_DNA.DR   PIR; E64992; E64992.DR   RefSeq; NP_416727.1; NC_000913.3.DR   RefSeq; WP_000580275.1; NZ_LN832404.1.DR   ProteinModelPortal; P76460; -.DR   BioGrid; 4261814; 7.DR   STRING; 316385.ECDH10B_2381; -.DR   TCDB; 2.A.73.1.1; the short chain fatty acid uptake (atoe) family.DR   PaxDb; P76460; -.DR   PRIDE; P76460; -.DR   EnsemblBacteria; AAC75283; AAC75283; b2223.DR   EnsemblBacteria; BAA16019; BAA16019; BAA16019.DR   GeneID; 946721; -.DR   KEGG; ecj:JW2217; -.DR   KEGG; eco:b2223; -.DR   PATRIC; fig|1411691.4.peg.12; -.DR   EchoBASE; EB1622; -.DR   EcoGene; EG11671; atoE.DR   eggNOG; ENOG4105CRW; Bacteria.DR   eggNOG; COG2031; LUCA.DR   HOGENOM; HOG000295152; -.DR   InParanoid; P76460; -.DR   KO; K02106; -.DR   OMA; QDKTIAH; -.DR   PhylomeDB; P76460; -.DR   BioCyc; EcoCyc:EG11671-MONOMER; -.DR   BioCyc; MetaCyc:EG11671-MONOMER; -.DR   PRO; PR:P76460; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0016021; C:integral component of membrane; IEA:UniProtKB-KW.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   InterPro; IPR006161; CHP00366.DR   InterPro; IPR006160; SCFA_transpt_AtoE.DR   PANTHER; PTHR41983; PTHR41983; 1.DR   Pfam; PF02667; SCFA_trans; 1.DR   TIGRFAMs; TIGR00366; TIGR00366; 1.PE   1: Evidence at protein level;KW   Cell inner membrane; Cell membrane; Complete proteome; Membrane;KW   Reference proteome; Transmembrane; Transmembrane helix; Transport.FT   CHAIN         1    440       Putative short-chain fatty acidFT                                transporter.FT                                /FTId=PRO_0000064733.FT   TOPO_DOM      1     19       Periplasmic. {ECO:0000255}.FT   TRANSMEM     20     40       Helical. {ECO:0000255}.FT   TOPO_DOM     41     53       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM     54     74       Helical. {ECO:0000255}.FT   TOPO_DOM     75    102       Periplasmic. {ECO:0000255}.FT   TRANSMEM    103    123       Helical. {ECO:0000255}.FT   TOPO_DOM    124    137       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    138    158       Helical. {ECO:0000255}.FT   TRANSMEM    159    179       Helical. {ECO:0000255}.FT   TOPO_DOM    180    180       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    181    201       Helical. {ECO:0000255}.FT   TOPO_DOM    202    244       Periplasmic. {ECO:0000255}.FT   TRANSMEM    245    265       Helical. {ECO:0000255}.FT   TRANSMEM    266    286       Helical. {ECO:0000255}.FT   TOPO_DOM    287    313       Periplasmic. {ECO:0000255}.FT   TRANSMEM    314    334       Helical. {ECO:0000255}.FT   TOPO_DOM    335    351       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    352    372       Helical. {ECO:0000255}.FT   TOPO_DOM    373    394       Periplasmic. {ECO:0000255}.FT   TRANSMEM    395    415       Helical. {ECO:0000255}.FT   TOPO_DOM    416    419       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM    420    440       Helical. {ECO:0000255}.SQ   SEQUENCE   440 AA;  47528 MW;  5C9D5ECE79F5FA71 CRC64;     MIGRISRFMT RFVSRWLPDP LIFAMLLTLL TFVIALWLTP QTPISMVKMW GDGFWNLLAF     GMQMALIIVT GHALASSAPV KSLLRTAASA AKTPVQGVML VTFFGSVACV INWGFGLVVG     AMFAREVARR VPGSDYPLLI ACAYIGFLTW GGGFSGSMPL LAATPGNPVE HIAGLIPVGD     TLFSGFNIFI TVALIVVMPF ITRMMMPKPS DVVSIDPKLL MEEADFQKQL PKDAPPSERL     EESRILTLII GALGIAYLAM YFSEHGFNIT INTVNLMFMI AGLLLHKTPM AYMRAISAAA     RSTAGILVQF PFYAGIQLMM EHSGLGGLIT EFFINVANKD TFPVMTFFSS ALINFAVPSG     GGHWVIQGPF VIPAAQALGA DLGKSVMAIA YGEQWMNMAQ PFWALPALAI AGLGVRDIMG     YCITALLFSG VIFVIGLTLF//\\n',\n",
       " ' ID   BAES_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ATP6_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01393, ECO:0000269|PubMed:15919996}; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01393,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   ATPD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01416, ECO:0000269|PubMed:16079137}; Peripheral membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01416,CC       ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   ATPZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   AZOR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BAER_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   BDCR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BETA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000269|PubMed:3512525};CC       Peripheral membrane protein {ECO:0000269|PubMed:3512525}.CC   \\n',\n",
       " ' ID   BGLA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BGLX_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   ATPE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:21778229};CC       Peripheral membrane protein {ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:21778229}.CC   \\n',\n",
       " ' ID   BAMC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00924, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00924,CC       ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BDCA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BEPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000255|HAMAP-Rule:MF_00997,CC       ECO:0000269|PubMed:22491786, ECO:0000269|PubMed:24003122}. Note=ACC       significant amount of BepA is membrane-associated. ThisCC       localization could result from interaction with the BAM complex.CC   \\n',\n",
       " ' ID   BGLB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BASR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   BCSB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Single-pass type ICC       membrane protein.CC   \\n',\n",
       " ' ID   BCSC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BCSG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Multi-CC       pass membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   BGA2_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BHSA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:22089859}.CC   \\n',\n",
       " ' ID   BIOD2_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00336}.CC   \\n',\n",
       " ' ID   BIOP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000269|Ref.5};CC       Multi-pass membrane protein {ECO:0000255}.CC   \\n',\n",
       " ' ID   BORD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000305}; Lipid-anchorCC       {ECO:0000305}.CC   \\n',\n",
       " ' ID   AZUC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000305|PubMed:19121005, ECO:0000305|PubMed:21778229};CC       Peripheral membrane protein {ECO:0000305|PubMed:19121005,CC       ECO:0000305|PubMed:21778229}.CC   \\n',\n",
       " ' ID   BAMB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00923, ECO:0000269|PubMed:15851030,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00923,CC       ECO:0000269|PubMed:15851030, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BAMD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00922, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00922,CC       ECO:0000269|PubMed:16079137, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BAX_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   BIOD1_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   BLC_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Lipid-anchor.CC   \\n',\n",
       " ' ID   BLUF_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BTSR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BTSS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000255}.CC   \\n',\n",
       " ' ID   ATPG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:16079137}; Peripheral membrane proteinCC       {ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   ATPL_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BAME_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00925, ECO:0000269|PubMed:17404237,CC       ECO:0000269|PubMed:26900875, ECO:0000269|PubMed:26901871}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_00925,CC       ECO:0000269|PubMed:17404237, ECO:0000269|PubMed:27686148}.CC   \\n',\n",
       " ' ID   BASS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   BCCP_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BDM_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   BGLG_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BICB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BIOC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BIRA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BARA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BCR_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BCSQP_ECOLI  \\n',\n",
       " '\\n',\n",
       " ' ID   BGAL_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BGLH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000305|PubMed:10027967}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:10027967}. Note=Can be easily removed from theCC       outer membrane by treatment with detergent.CC   \\n',\n",
       " ' ID   BGLJ_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BCP_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   BCSA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Multi-CC       pass membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   BCSE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BCSF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Single-CC       pass membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   BCSQ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:19400787}.CC       Note=Localizes at the cell pole, this does not require any genesCC       of the divergently expressed yhjRbcsQABZC and bcsEFG operons.CC       {ECO:0000269|PubMed:19400787}.CC   \\n',\n",
       " ' ID   BEEE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BETT_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   BFD_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   BGLR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BIOH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250}.CC   \\n',\n",
       " ' ID   BOLA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BSSR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   BTST_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:22685278,CC       ECO:0000269|PubMed:29061664}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:29061664}.CC   \\n',\n",
       " ' ID   MENF_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MENH_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MEPM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Single-CC       pass membrane protein {ECO:0000305}. Note=Uniform peripheralCC       location, with partial enrichment at cell poles.CC   \\n',\n",
       " ' ID   METAS_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00295,CC       ECO:0000305}.CC   \\n',\n",
       " ' ID   METJ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MIOC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MLAF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000305|PubMed:19383799}; Peripheral membrane proteinCC       {ECO:0000305|PubMed:19383799}; Cytoplasmic sideCC       {ECO:0000305|PubMed:19383799}.CC   \\n',\n",
       " ' ID   MMUM_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MNGR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MNMG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:17062623}.CC   \\n',\n",
       " ' ID   MELR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MEND_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MEPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305|PubMed:15292190}.CC   \\n',\n",
       " ' ID   METK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00086}.CC   \\n',\n",
       " ' ID   MLAB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305|PubMed:19383799}.CC   \\n',\n",
       " ' ID   MLAC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305|PubMed:19383799}.CC   \\n',\n",
       " ' ID   MNMA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MOBA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   KDGT_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   KDPB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00285, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2849541}; Multi-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_00285, ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   KDSC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   KDSD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   KDTA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000305|PubMed:1577828}; Single-pass membrane proteinCC       {ECO:0000305|PubMed:1577828}; Cytoplasmic sideCC       {ECO:0000305|PubMed:1577828}.CC   \\n',\n",
       " ' ID   KDUD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   KEFB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01412, ECO:0000269|PubMed:15919996}; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01412,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   KEFC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01413, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2046548}; Multi-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_01413, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2046548}.CC   \\n',\n",
       " ' ID   KDPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00275, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:2849541, ECO:0000269|PubMed:7896809}; Multi-CC       pass membrane protein {ECO:0000255|HAMAP-Rule:MF_00275,CC       ECO:0000269|PubMed:7896809}.CC   \\n',\n",
       " ' ID   KDPE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:1532388}.CC   \\n',\n",
       " ' ID   KGUA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   KILR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   KITH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00124}.CC   \\n',\n",
       " ' ID   KPYK1_ECOLI  \\n',\n",
       " '\\n',\n",
       " ' ID   KPYK2_ECOLI  \\n',\n",
       " '\\n',\n",
       " ' ID   LACI_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPMG_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPTB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:16079137}.CC       Cell inner membrane {ECO:0000269|PubMed:16079137}; PeripheralCC       membrane protein {ECO:0000269|PubMed:16079137}; Cytoplasmic sideCC       {ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   LSRG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_02051}.CC   \\n',\n",
       " ' ID   LSRR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   LTND_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LYSR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; J01614; AAA83862.1; -; Genomic_DNA.DR   EMBL; U29581; AAB40486.1; -; Genomic_DNA.DR   EMBL; U00096; AAC75878.1; -; Genomic_DNA.DR   EMBL; AP009048; BAE76908.1; -; Genomic_DNA.DR   EMBL; J03732; AAA23471.1; -; Genomic_DNA.DR   PIR; A03565; RGECK.DR   RefSeq; NP_417316.1; NC_000913.3.DR   RefSeq; WP_000741825.1; NZ_LN832404.1.DR   ProteinModelPortal; P03030; -.DR   SMR; P03030; -.DR   BioGrid; 4262309; 10.DR   DIP; DIP-10135N; -.DR   IntAct; P03030; 6.DR   STRING; 316385.ECDH10B_3011; -.DR   PaxDb; P03030; -.DR   PRIDE; P03030; -.DR   EnsemblBacteria; AAC75878; AAC75878; b2839.DR   EnsemblBacteria; BAE76908; BAE76908; BAE76908.DR   GeneID; 947311; -.DR   KEGG; ecj:JW2807; -.DR   KEGG; eco:b2839; -.DR   PATRIC; fig|1411691.4.peg.3895; -.DR   EchoBASE; EB0546; -.DR   EcoGene; EG10551; lysR.DR   eggNOG; ENOG4105RQF; Bacteria.DR   eggNOG; ENOG410XT7K; LUCA.DR   HOGENOM; HOG000233527; -.DR   InParanoid; P03030; -.DR   OMA; FHAVMTT; -.DR   PhylomeDB; P03030; -.DR   BioCyc; EcoCyc:PD00360; -.DR   PRO; PR:P03030; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0005737; C:cytoplasm; IEA:UniProtKB-SubCell.DR   GO; GO:0003677; F:DNA binding; IEA:UniProtKB-KW.DR   GO; GO:0003700; F:DNA binding transcription factor activity; IEA:InterPro.DR   GO; GO:0009089; P:lysine biosynthetic process via diaminopimelate; IDA:EcoCyc.DR   GO; GO:0010628; P:positive regulation of gene expression; IMP:EcoCyc.DR   GO; GO:0006351; P:transcription, DNA-templated; IEA:UniProtKB-KW.DR   CDD; cd08456; PBP2_LysR; 1.DR   Gene3D; 1.10.10.10; -; 1.DR   InterPro; IPR037414; LysR_PBP2.DR   InterPro; IPR005119; LysR_subst-bd.DR   InterPro; IPR000847; Tscrpt_reg_HTH_LysR.DR   InterPro; IPR036388; WH-like_DNA-bd_sf.DR   InterPro; IPR036390; WH_DNA-bd_sf.DR   Pfam; PF00126; HTH_1; 1.DR   Pfam; PF03466; LysR_substrate; 1.DR   PRINTS; PR00039; HTHLYSR.DR   SUPFAM; SSF46785; SSF46785; 1.DR   PROSITE; PS50931; HTH_LYSR; 1.PE   4: Predicted;KW   Activator; Complete proteome; Cytoplasm; DNA-binding;KW   Reference proteome; Repressor; Transcription;KW   Transcription regulation.FT   CHAIN         1    311       Transcriptional activator protein LysR.FT                                /FTId=PRO_0000105669.FT   DOMAIN        1     61       HTH lysR-type. {ECO:0000255|PROSITE-FT                                ProRule:PRU00253}.FT   DNA_BIND     21     40       H-T-H motif. {ECO:0000255|PROSITE-FT                                ProRule:PRU00253}.SQ   SEQUENCE   311 AA;  34365 MW;  3E1913BB6A2CF8F9 CRC64;     MAAVNLRHIE IFHAVMTAGS LTEAAHLLHT SQPTVSRELA RFEKVIGLKL FERVRGRLHP     TVQGLRLFEE VQRSWYGLDR IVSAAESLRE FRQGELSIAC LPVFSQSFLP QLLQPFLARY     PDVSLNIVPQ ESPLLEEWLS AQRHDLGLTE TLHTPAGTER TELLSLDEVC VLPPGHPLAV     KKVLTPDDFQ GENYISLSRT DSYRQLLDQL FTEHQVKRRM IVETHSAASV CAMVRAGVGI     SVVNPLTALD YAASGLVVRR FSIAVPFTVS LIRPLHRPSS ALVQAFSGHL QAGLPKLVTS     LDAILSSATT A//\\n',\n",
       " ' ID   MARR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MASZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MIAB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01864}.CC   \\n',\n",
       " ' ID   MNGA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|PROSITE-CC       ProRule:PRU00427, ECO:0000305|PubMed:14645248}; Multi-passCC       membrane protein {ECO:0000255|PROSITE-ProRule:PRU00427,CC       ECO:0000305|PubMed:14645248}.CC   \\n',\n",
       " ' ID   MNMC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   MNTH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00221, ECO:0000269|PubMed:14607838}; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_00221,CC       ECO:0000269|PubMed:14607838}.CC   \\n',\n",
       " ' ID   MPL_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Secreted.CC   \\n',\n",
       " ' ID   KCY_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   KDPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00276, ECO:0000269|PubMed:2849541}; Single-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_00276}.CC   \\n',\n",
       " ' ID   KDSB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00057}.CC   \\n',\n",
       " ' ID   MLIC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:18369469}; Lipid-anchor {ECO:0000255|PROSITE-CC       ProRule:PRU00303, ECO:0000269|PubMed:18369469}. Note=Anchored toCC       the periplasmic side. {ECO:0000269|PubMed:18369469}.CC   \\n',\n",
       " ' ID   MLTA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Lipid-anchor.CC   \\n',\n",
       " ' ID   MLTB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Lipid-anchor;CC       Periplasmic side.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; U18785; AAB60060.1; -; Genomic_DNA.DR   EMBL; U00096; AAC75743.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA16563.1; -; Genomic_DNA.DR   EMBL; J02708; -; NOT_ANNOTATED_CDS; Genomic_DNA.DR   PIR; A65050; A65050.DR   RefSeq; NP_417181.1; NC_000913.3.DR   RefSeq; WP_001295177.1; NZ_LN832404.1.DR   PDB; 1D0K; X-ray; 2.02 A; A=40-361.DR   PDB; 1D0L; X-ray; 1.97 A; A=40-361.DR   PDB; 1D0M; X-ray; 2.47 A; A=40-361.DR   PDB; 1LTM; X-ray; 1.70 A; A=42-361.DR   PDB; 1QDR; X-ray; 2.10 A; A=40-361.DR   PDB; 1QDT; X-ray; 2.10 A; A=40-361.DR   PDB; 1QUS; X-ray; 1.70 A; A=40-361.DR   PDB; 1QUT; X-ray; 2.44 A; A=40-361.DR   PDBsum; 1D0K; -.DR   PDBsum; 1D0L; -.DR   PDBsum; 1D0M; -.DR   PDBsum; 1LTM; -.DR   PDBsum; 1QDR; -.DR   PDBsum; 1QDT; -.DR   PDBsum; 1QUS; -.DR   PDBsum; 1QUT; -.DR   ProteinModelPortal; P41052; -.DR   SMR; P41052; -.DR   BioGrid; 4262079; 390.DR   DIP; DIP-10221N; -.DR   IntAct; P41052; 10.DR   STRING; 316385.ECDH10B_2869; -.DR   BindingDB; P41052; -.DR   ChEMBL; CHEMBL2046262; -.DR   DrugBank; DB03709; Bicine.DR   DrugBank; DB02595; Bulgecin A.DR   CAZy; GH103; Glycoside Hydrolase Family 103.DR   PaxDb; P41052; -.DR   PRIDE; P41052; -.DR   EnsemblBacteria; AAC75743; AAC75743; b2701.DR   EnsemblBacteria; BAA16563; BAA16563; BAA16563.DR   GeneID; 947184; -.DR   KEGG; ecj:JW2671; -.DR   KEGG; eco:b2701; -.DR   PATRIC; fig|511145.12.peg.2792; -.DR   EchoBASE; EB2561; -.DR   EcoGene; EG12699; mltB.DR   eggNOG; ENOG4105CVE; Bacteria.DR   eggNOG; COG2951; LUCA.DR   HOGENOM; HOG000257828; -.DR   InParanoid; P41052; -.DR   KO; K08305; -.DR   OMA; SSIRDYA; -.DR   PhylomeDB; P41052; -.DR   BioCyc; EcoCyc:G7410-MONOMER; -.DR   BioCyc; MetaCyc:G7410-MONOMER; -.DR   EvolutionaryTrace; P41052; -.DR   PRO; PR:P41052; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0009279; C:cell outer membrane; IDA:EcoCyc.DR   GO; GO:0030288; C:outer membrane-bounded periplasmic space; TAS:EcoCyc.DR   GO; GO:0005509; F:calcium ion binding; IDA:EcoCyc.DR   GO; GO:0016829; F:lyase activity; IEA:UniProtKB-KW.DR   GO; GO:0008932; F:lytic endotransglycosylase activity; IDA:EcoCyc.DR   GO; GO:0008933; F:lytic transglycosylase activity; IDA:EcoCyc.DR   GO; GO:0031402; F:sodium ion binding; IDA:EcoCyc.DR   GO; GO:0071555; P:cell wall organization; IEA:UniProtKB-KW.DR   GO; GO:0009253; P:peptidoglycan catabolic process; IDA:EcoCyc.DR   InterPro; IPR023346; Lysozyme-like_dom_sf.DR   InterPro; IPR011757; Lytic_transglycosylase_MltB.DR   InterPro; IPR031304; SLT_2.DR   PANTHER; PTHR30163:SF0; PTHR30163:SF0; 1.DR   Pfam; PF13406; SLT_2; 1.DR   SUPFAM; SSF53955; SSF53955; 1.DR   TIGRFAMs; TIGR02282; MltB; 1.DR   PROSITE; PS51257; PROKAR_LIPOPROTEIN; 1.PE   1: Evidence at protein level;KW   3D-structure; Cell outer membrane; Cell wall biogenesis/degradation;KW   Complete proteome; Lipoprotein; Lyase; Membrane; Palmitate;KW   Reference proteome; Signal.FT   SIGNAL        1     18       {ECO:0000305}.FT   CHAIN        19    361       Membrane-bound lytic mureinFT                                transglycosylase B.FT                                /FTId=PRO_0000032784.FT   ACT_SITE    162    162FT   LIP\\n',\n",
       " ' ID   MLTD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000305}; Lipid-anchorCC       {ECO:0000305}.CC   \\n',\n",
       " ' ID   MNTR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   MODF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000250};CC       Peripheral membrane protein {ECO:0000250}.CC   \\n',\n",
       " ' ID   MGLC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MIND_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Peripheral membraneCC       protein.CC   \\n',\n",
       " ' ID   MINE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MLAA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000305|PubMed:19383799}; Lipid-anchor {ECO:0000255|PROSITE-CC       ProRule:PRU00303, ECO:0000305|PubMed:19383799}.CC   \\n',\n",
       " ' ID   MLC_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   MLRA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MHPT_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:23934492}; Multi-CC       pass membrane protein {ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:23934492}.CC   \\n',\n",
       " ' ID   MIAA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MIPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   MNAT_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MNGB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOAB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOAD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOBB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOG_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   CODB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   COF_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   CPDA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   CSPE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250}.CC   \\n',\n",
       " ' ID   KAD_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00235}.CC   \\n',\n",
       " ' ID   KBL_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   KHSE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   OTSA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PA1_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:6397463}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:6397463}. Note=One of the very few enzymesCC       located there.CC   \\n',\n",
       " ' ID   PAAJ_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PANF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   PARC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.CC   \\n',\n",
       " ' ID   PBL_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   PBP7_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   PBPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:7006606}; Single-pass type II membrane proteinCC       {ECO:0000255}.CC   \\n',\n",
       " ' ID   OTNK_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   OTSB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PAAA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PAGP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_00837, ECO:0000269|PubMed:11013210}.CC   \\n',\n",
       " ' ID   PAL_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_02204, ECO:0000269|PubMed:1574003}; Lipid-anchorCC       {ECO:0000255|HAMAP-Rule:MF_02204, ECO:0000269|PubMed:1574003}.CC       Note=Accumulates at cell constriction sites. Recruitment to theCC       division site is dependent on FtsN activity.CC       {ECO:0000269|PubMed:17233825}.CC   \\n',\n",
       " ' ID   PANE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   PANZ_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NRFE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000305|PubMed:15919996}.CC   \\n',\n",
       " ' ID   NUOA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01394, ECO:0000269|PubMed:15919996}; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01394,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   NUOI_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305};CC       Peripheral membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   NUOK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NUON_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   OGRK_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   OGT_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00772}.CC   \\n',\n",
       " ' ID   OPPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   OMPR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   OMPW_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane.CC   \\n',\n",
       " ' ID   OMPX_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   OPPB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ORN_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   PANC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   NUOH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NUOM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NUPX_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000255}.CC   \\n',\n",
       " ' ID   NUSA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00945,CC       ECO:0000269|PubMed:17272300}. Note=Colocalizes with nucleoids.CC   \\n',\n",
       " ' ID   OMPT_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NSRR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NUOE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NUOF_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NUOG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:16079137}.CC       Cell inner membrane {ECO:0000269|PubMed:16079137}; PeripheralCC       membrane protein {ECO:0000269|PubMed:16079137}.CC   \\n',\n",
       " ' ID   NUPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000255}.CC   \\n',\n",
       " ' ID   NUPG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_02049, ECO:0000269|PubMed:15513740,CC       ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:3311747}; Multi-CC       pass membrane protein {ECO:0000255|HAMAP-Rule:MF_02049}.CC   \\n',\n",
       " ' ID   NUSB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NUSG_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ODO1_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   OMPG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Multi-pass membraneCC       protein.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; U49400; AAC34720.1; -; Genomic_DNA.DR   EMBL; U00096; AAC74401.1; -; Genomic_DNA.DR   EMBL; AP009048; BAE76401.1; -; Genomic_DNA.DR   PIR; B64881; B64881.DR   RefSeq; NP_415835.1; NC_000913.3.DR   RefSeq; WP_000735257.1; NZ_LN832404.1.DR   PDB; 2F1C; X-ray; 2.30 A; X=22-301.DR   PDB; 2IWV; X-ray; 2.30 A; A/B/C/D=22-301.DR   PDB; 2IWW; X-ray; 2.70 A; A/B=22-301.DR   PDB; 2JQY; NMR; -; A=22-301.DR   PDB; 2WVP; X-ray; 2.40 A; A=22-301.DR   PDB; 2X9K; X-ray; 2.18 A; A=22-301.DR   PDB; 4CTD; X-ray; 3.20 A; A/B=22-301.DR   PDB; 5MWV; NMR; -; A=22-301.DR   PDBsum; 2F1C; -.DR   PDBsum; 2IWV; -.DR   PDBsum; 2IWW; -.DR   PDBsum; 2JQY; -.DR   PDBsum; 2WVP; -.DR   PDBsum; 2X9K; -.DR   PDBsum; 4CTD; -.DR   PDBsum; 5MWV; -.DR   ProteinModelPortal; P76045; -.DR   SMR; P76045; -.DR   BioGrid; 4260146; 153.DR   DIP; DIP-10399N; -.DR   IntAct; P76045; 1.DR   STRING; 316385.ECDH10B_1438; -.DR   DrugBank; DB04233; (Hydroxyethyloxy)Tri(Ethyloxy)Octane.DR   DrugBank; DB04147; Lauryl Dimethylamine-N-Oxide.DR   TCDB; 1.B.21.1.1; the ompg porin (ompg) family.DR   PaxDb; P76045; -.DR   PRIDE; P76045; -.DR   EnsemblBacteria; AAC74401; AAC74401; b1319.DR   EnsemblBacteria; BAE76401; BAE76401; BAE76401.DR   GeneID; 945889; -.DR   KEGG; ecj:JW1312; -.DR   KEGG; eco:b1319; -.DR   PATRIC; fig|1411691.4.peg.959; -.DR   EchoBASE; EB3202; -.DR   EcoGene; EG13428; ompG.DR   eggNOG; ENOG4107EFV; Bacteria.DR   eggNOG; ENOG410YHNX; LUCA.DR   HOGENOM; HOG000120098; -.DR   KO; K14053; -.DR   OMA; PSIYFNA; -.DR   BioCyc; EcoCyc:G6657-MONOMER; -.DR   EvolutionaryTrace; P76045; -.DR   PRO; PR:P76045; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0009279; C:cell outer membrane; IDA:EcoCyc.DR   GO; GO:0045203; C:integral component of cell outer membrane; IDA:EcoCyc.DR   GO; GO:0046930; C:pore complex; IEA:UniProtKB-KW.DR   GO; GO:0015481; F:maltose transporting porin activity; IDA:EcoCyc.DR   GO; GO:0015478; F:oligosaccharide transporting porin activity; IDA:EcoCyc.DR   GO; GO:0015288; F:porin activity; IMP:EcoliWiki.DR   GO; GO:0034219; P:carbohydrate transmembrane transport; IDA:EcoCyc.DR   GO; GO:0006811; P:ion transport; IEA:UniProtKB-KW.DR   InterPro; IPR018981; Outer_membrane_protein_G.DR   Pfam; PF09381; Porin_OmpG; 1.PE   1: Evidence at protein level;KW   3D-structure; Cell outer membrane; Complete proteome;KW   Direct protein sequencing; Ion transport; Membrane; Porin;KW   Reference proteome; Signal; Transmembrane; Transmembrane beta strand;KW   Transport.FT   SIGNAL        1     21       {ECO:0000269|PubMed:9721282}.FT   CHAIN        22    301       Outer membrane protein G.FT                                /FTId=PRO_0000025198.FT   TRANSMEM     27     35       Beta stranded. {ECO:0000255}.FT   TRANSMEM     47     57       Beta stranded. {ECO:0000255}.FT   TRANSMEM     64     72       Beta stranded. {ECO:0000255}.FT   TRANSMEM     89     98       Beta stranded. {ECO:0000255}.FT   TRANSMEM    104    112       Beta stranded. {ECO:0000255}.FT   TRANSMEM    129    136       Beta stranded. {ECO:0000255}.FT   TRANSMEM    149    158       Beta stranded. {ECO:0000255}.FT   TRANSMEM    172    182       Beta stranded. {ECO:0000255}.FT   TRANSMEM    186    195       Beta stranded. {ECO:0000255}.FT   TRANSMEM    201    209       Beta stranded. {ECO:0000255}.FT   TRANSMEM    213    222       Beta stranded. {ECO:0000255}.FT   TRANSMEM    230    238       Beta stranded. {ECO:0000255}.FT   TRANSMEM    240    248       Beta stranded. {ECO:0000255}.FT   TRANSMEM    254    265       Beta stranded. {ECO:0000255}.FT   TRANSMEM    269    279       Beta stranded. {ECO:0000255}.FT   TRANSMEM    289    300       Beta stranded. {ECO:0000255}.FT   STRAND       24     26       {ECO:0000244|PDB:2JQY}.FT   STRAND       28     42       {ECO:0000244|PDB:2X9K}.FT   STRAND       45     60       {ECO:0000244|PDB:2X9K}.FT   STRAND       62     75       {ECO:0000244|PDB:2X9K}.FT   STRAND       78     81       {ECO:0000244|PDB:2X9K}.FT   TURN         82     84       {ECO:0000244|PDB:5MWV}.FT   STRAND       86    118       {ECO:0000244|PDB:2X9K}.FT   STRAND      127    160       {ECO:0000244|PDB:2X9K}.FT   HELIX       162    165       {ECO:0000244|PDB:2X9K}.FT   STRAND      169    181       {ECO:0000244|PDB:2X9K}.FT   STRAND      183    198       {ECO:0000244|PDB:2X9K}.FT   STRAND      200    203       {ECO:0000244|PDB:2F1C}.FT   STRAND      209    217       {ECO:0000244|PDB:2X9K}.FT   STRAND      220    222       {ECO:0000244|PDB:2X9K}.FT   STRAND      225    239       {ECO:0000244|PDB:2X9K}.FT   STRAND      243    247       {ECO:0000244|PDB:2X9K}.FT   STRAND      254    264       {ECO:0000244|PDB:2X9K}.FT   STRAND      266    280       {ECO:0000244|PDB:2X9K}.FT   TURN        283    285       {ECO:0000244|PDB:2WVP}.FT   STRAND      289    300       {ECO:0000244|PDB:2X9K}.SQ   SEQUENCE   301 AA;  34913 MW;  63D1234F3DCCFECF CRC64;     MKKLLPCTAL VMCAGMACAQ AEERNDWHFN IGAMYEIENV EGYGEDMDGL AEPSVYFNAA     NGPWRIALAY YQEGPVDYSA GKRGTWFDRP ELEVHYQFLE NDDFSFGLTG GFRNYGYHYV     DEPGKDTANM QRWKIAPDWD VKLTDDLRFN GWLSMYKFAN DLNTTGYADT RVETETGLQY     TFNETVALRV NYYLERGFNM DDSRNNGEFS TQEIRAYLPL TLGNHSVTPY TRIGLDRWSN     WDWQDDIERE GHDFNRVGLF YGYDFQNGLS VSLEYAFEWQ DHDEGDSDKF HYAGVGVNYS     F//\\n',\n",
       " ' ID   OMPN_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   OPGE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   OPGH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   ORTT_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000305|PubMed:25643179}; Multi-pass membrane proteinCC       {ECO:0000305}.CC   \\n',\n",
       " ' ID   OSMC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   OMPL_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:11080145}.CC   \\n',\n",
       " ' ID   OSMY_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   PAAD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PAAG_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PAAI_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PAAK_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PABA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   PABC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   ODP2_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   OPGB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   OPPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   OPPD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Peripheral membraneCC       protein.CC   \\n',\n",
       " ' ID   OTNC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   OXC_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   OXYR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MTOX_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MURB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   NADA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NAGC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NAGD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NAGK_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MURE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   NADE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NANS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305|PubMed:19749043}.CC   \\n',\n",
       " ' ID   NAPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000255|HAMAP-Rule:MF_01630,CC       ECO:0000269|PubMed:10234835}.CC   \\n',\n",
       " ' ID   NAPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Single-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NADB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NADR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.CC       Cytoplasm {ECO:0000250}.CC   \\n',\n",
       " ' ID   NANM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:18063573}.CC   \\n',\n",
       " ' ID   NAPB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:10234835,CC       ECO:0000269|PubMed:10548535, ECO:0000269|PubMed:17130127}.CC   \\n',\n",
       " ' ID   NAPF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:17074894}.CC       Note=Loosely attached to the inner side of the membrane.CC       {ECO:0000269|PubMed:17074894}.CC   \\n',\n",
       " ' ID   NAPH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:14674886, ECO:0000269|PubMed:15919996}; Multi-CC       pass membrane protein {ECO:0000269|PubMed:14674886}.CC   \\n',\n",
       " ' ID   NARK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   NFUA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NHAP2_ECOLI  \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01075}; Multi-pass membrane protein {ECO:0000255|HAMAP-CC       Rule:MF_01075}.CC   \\n',\n",
       " ' ID   NHOA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250|UniProtKB:Q00267}.CC   \\n',\n",
       " ' ID   MZRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00904, ECO:0000269|PubMed:19432797,CC       ECO:0000269|PubMed:20889743}; Single-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_00904, ECO:0000269|PubMed:19432797,CC       ECO:0000269|PubMed:20889743}.CC   \\n',\n",
       " ' ID   NARP_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NARQ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; M94724; AAA24202.1; -; Genomic_DNA.DR   EMBL; X65714; CAA46630.1; -; Genomic_DNA.DR   EMBL; U00096; AAC75522.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA16343.1; -; Genomic_DNA.DR   EMBL; U12598; AAA20583.1; -; Genomic_DNA.DR   PIR; D65022; D65022.DR   RefSeq; NP_416964.1; NC_000913.3.DR   RefSeq; WP_001300881.1; NZ_CP014272.1.DR   PDB; 5IJI; X-ray; 1.94 A; A=1-230.DR   PDB; 5JEF; X-ray; 2.42 A; A/B=1-230.DR   PDB; 5JEQ; X-ray; 1.90 A; A=1-230.DR   PDB; 5JGP; X-ray; 2.70 A; A=1-230.DR   PDBsum; 5IJI; -.DR   PDBsum; 5JEF; -.DR   PDBsum; 5JEQ; -.DR   PDBsum; 5JGP; -.DR   ProteinModelPortal; P27896; -.DR   SMR; P27896; -.DR   BioGrid; 4260928; 11.DR   DIP; DIP-10318N; -.DR   IntAct; P27896; 12.DR   STRING; 316385.ECDH10B_2634; -.DR   iPTMnet; P27896; -.DR   PaxDb; P27896; -.DR   PRIDE; P27896; -.DR   EnsemblBacteria; AAC75522; AAC75522; b2469.DR   EnsemblBacteria; BAA16343; BAA16343; BAA16343.DR   GeneID; 946948; -.DR   KEGG; ecj:JW2453; -.DR   KEGG; eco:b2469; -.DR   PATRIC; fig|511145.12.peg.2563; -.DR   EchoBASE; EB1429; -.DR   EcoGene; EG11460; narQ.DR   eggNOG; ENOG4105BZU; Bacteria.DR   eggNOG; ENOG410XNMH; LUCA.DR   HOGENOM; HOG000275465; -.DR   InParanoid; P27896; -.DR   KO; K07674; -.DR   OMA; FLQIQLT; -.DR   PhylomeDB; P27896; -.DR   BioCyc; EcoCyc:NARQ-MONOMER; -.DR   BRENDA; 2.7.13.3; 2026.DR   PRO; PR:P27896; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0016021; C:integral component of membrane; IEA:UniProtKB-KW.DR   GO; GO:0005622; C:intracellular; IEA:GOC.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   GO; GO:0005524; F:ATP binding; IEA:UniProtKB-KW.DR   GO; GO:0000155; F:phosphorelay sensor kinase activity; IMP:EcoCyc.DR   GO; GO:0046983; F:protein dimerization activity; IEA:InterPro.DR   GO; GO:0071249; P:cellular response to nitrate; IMP:EcoCyc.DR   GO; GO:0071250; P:cellular response to nitrite; IMP:EcoCyc.DR   GO; GO:0042128; P:nitrate assimilation; IEA:UniProtKB-KW.DR   GO; GO:0000160; P:phosphorelay signal transduction system; IMP:EcoCyc.DR   CDD; cd06225; HAMP; 1.DR   CDD; cd00075; HATPase_c; 1.DR   Gene3D; 3.30.565.10; -; 1.DR   InterPro; IPR003660; HAMP_dom.DR   InterPro; IPR003594; HATPase_C.DR   InterPro; IPR036890; HATPase_C_sf.DR   InterPro; IPR005467; His_kinase_dom.DR   InterPro; IPR029095; NarX-like_N.DR   InterPro; IPR016380; Sig_transdc_His_kin_NarX/NarQ.DR   InterPro; IPR011712; Sig_transdc_His_kin_sub3_dim/P.DR   Pfam; PF00672; HAMP; 1.DR   Pfam; PF02518; HATPase_c; 1.DR   Pfam; PF07730; HisKA_3; 1.DR   Pfam; PF13675; PilJ; 1.DR   PIRSF; PIRSF003167; STHK_NarX/NarQ; 1.DR   SMART; SM00304; HAMP; 1.DR   SMART; SM00387; HATPase_c; 1.DR   SUPFAM; SSF55874; SSF55874; 1.DR   PROSITE; PS50885; HAMP; 1.DR   PROSITE; PS50109; HIS_KIN; 1.PE   1: Evidence at protein level;KW   3D-structure; ATP-binding; Cell inner membrane; Cell membrane;KW   Complete proteome; Kinase; Membrane; Nitrate assimilation;KW   Nucleotide-binding; Phosphoprotein; Reference proteome; Transferase;KW   Transmembrane; Transmembrane helix; Two-component regulatory system.FT   CHAIN         1    566       Nitrate/nitrite sensor protein NarQ.FT                                /FTId=PRO_0000074810.FT   TOPO_DOM      1     13       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM     14     34       Helical. {ECO:0000255}.FT   TOPO_DOM     35    146       Periplasmic. {ECO:0000255}.FT   TRANSMEM    147    167       Helical. {ECO:0000255}.FT   TOPO_DOM    168    566       Cytoplasmic. {ECO:0000255}.FT   DOMAIN      174    227       HAMP. {ECO:0000255|PROSITE-FT                                ProRule:PRU00102}.FT   DOMAIN      364    559       Histidine kinase. {ECO:0000255|PROSITE-FT                                ProRule:PRU00107}.FT   MOD_RES     370    370       Phosphohistidine; by autocatalysis.FT                                {ECO:0000255|PROSITE-ProRule:PRU00107}.FT   HELIX         7     62       {ECO:0000244|PDB:5JEQ}.FT   HELIX        67     78       {ECO:0000244|PDB:5JEQ}.FT   HELIX        81     84       {ECO:0000244|PDB:5JEQ}.FT   HELIX        93    114       {ECO:0000244|PDB:5JEQ}.FT   HELIX       118    175       {ECO:0000244|PDB:5JEQ}.FT   HELIX       177    191       {ECO:0000244|PDB:5JEQ}.FT   HELIX       207    227       {ECO:0000244|PDB:5JEQ}.SQ   SEQUENCE   566 AA;  63697 MW;  1D08EA5BD328ACE2 CRC64;     MIVKRPVSAS LARAFFYIVL LSILSTGIAL LTLASSLRDA EAINIAGSLR MQSYRLGYDL     QSGSPQLNAH RQLFQQALHS PVLTNLNVWY VPEAVKTRYA HLNANWLEMN NRLSKGDLPW     YQANINNYVN QIDLFVLALQ HYAERKMLLV VAISLAGGIG IFTLVFFTLR RIRHQVVAPL     NQLVTASQRI EHGQFDSPPL DTNLPNELGL LAKTFNQMSS ELHKLYRSLE ASVEEKTRDL     HEAKRRLEVL YQCSQALNTS QIDVHCFRHI LQIVRDNEAA EYLELNVGEN WRISEGQPNP     ELPMQILPVT MQETVYGELH WQNSHVSSSE PLLNSVSSML GRGLYFNQAQ KHFQQLLLME     ERATIARELH DSLAQVLSYL RIQLTLLKRS IPEDNATAQS IMADFSQALN DAYRQLRELL     TTFRLTLQQA DLPSALREML DTLQNQTSAK LTLDCRLPTL ALDAQMQVHL LQIIREAVLN     AMKHANASEI AVSCVTAPDG NHTVYIRDNG IGIGEPKEPE GHYGLNIMRE RAERLGGTLT     FSQPSGGGTL VSISFRSAEG EESQLM//\\n',\n",
       " ' ID   NARW_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000250}.CC   \\n',\n",
       " ' ID   NCPP_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NDK_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00451,CC       ECO:0000305}.CC   \\n',\n",
       " ' ID   NADC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NADK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00361}.CC   \\n',\n",
       " ' ID   NAGZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NARG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.CC   \\n',\n",
       " ' ID   NARH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.CC   \\n',\n",
       " ' ID   NARI_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NARL_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NARX_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; X13360; CAA31741.1; -; Genomic_DNA.DR   EMBL; U00096; AAC74306.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA36090.1; -; Genomic_DNA.DR   EMBL; X15996; CAA34125.1; -; Genomic_DNA.DR   EMBL; M24910; AAA24198.1; -; Genomic_DNA.DR   EMBL; X65715; CAA46631.1; -; Genomic_DNA.DR   EMBL; X69189; CAA48934.1; -; Genomic_DNA.DR   PIR; S26137; RGECNX.DR   RefSeq; NP_415740.1; NC_000913.3.DR   RefSeq; WP_000918073.1; NZ_LN832404.1.DR   PDB; 3EZH; X-ray; 1.70 A; A/B=38-151.DR   PDB; 3EZI; X-ray; 1.70 A; A/B/C/D=42-148.DR   PDBsum; 3EZH; -.DR   PDBsum; 3EZI; -.DR   ProteinModelPortal; P0AFA2; -.DR   SMR; P0AFA2; -.DR   BioGrid; 4263270; 17.DR   DIP; DIP-35785N; -.DR   IntAct; P0AFA2; 5.DR   STRING; 316385.ECDH10B_1281; -.DR   PaxDb; P0AFA2; -.DR   PRIDE; P0AFA2; -.DR   EnsemblBacteria; AAC74306; AAC74306; b1222.DR   EnsemblBacteria; BAA36090; BAA36090; BAA36090.DR   GeneID; 945788; -.DR   KEGG; ecj:JW1213; -.DR   KEGG; eco:b1222; -.DR   PATRIC; fig|1411691.4.peg.1060; -.DR   EchoBASE; EB0640; -.DR   EcoGene; EG10646; narX.DR   eggNOG; ENOG4105BZU; Bacteria.DR   eggNOG; ENOG410XNMH; LUCA.DR   HOGENOM; HOG000275465; -.DR   InParanoid; P0AFA2; -.DR   KO; K07673; -.DR   OMA; NHYGLII; -.DR   PhylomeDB; P0AFA2; -.DR   BioCyc; EcoCyc:NARX-MONOMER; -.DR   BRENDA; 2.7.13.3; 2026.DR   EvolutionaryTrace; P0AFA2; -.DR   PRO; PR:P0AFA2; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0016021; C:integral component of membrane; IEA:UniProtKB-KW.DR   GO; GO:0005622; C:intracellular; IEA:GOC.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   GO; GO:0005524; F:ATP binding; IEA:UniProtKB-KW.DR   GO; GO:0004721; F:phosphoprotein phosphatase activity; IDA:EcoCyc.DR   GO; GO:0000155; F:phosphorelay sensor kinase activity; IDA:EcoCyc.DR   GO; GO:0046983; F:protein dimerization activity; IEA:InterPro.DR   GO; GO:0006974; P:cellular response to DNA damage stimulus; IEP:EcoliWiki.DR   GO; GO:0071249; P:cellular response to nitrate; IEP:EcoCyc.DR   GO; GO:0071250; P:cellular response to nitrite; IMP:EcoCyc.DR   GO; GO:0042128; P:nitrate assimilation; IEA:UniProtKB-KW.DR   GO; GO:0000160; P:phosphorelay signal transduction system; IDA:EcoCyc.DR   GO; GO:0046777; P:protein autophosphorylation; IDA:EcoCyc.DR   CDD; cd06225; HAMP; 1.DR   CDD; cd00075; HATPase_c; 1.DR   Gene3D; 3.30.565.10; -; 1.DR   InterPro; IPR003660; HAMP_dom.DR   InterPro; IPR003594; HATPase_C.DR   InterPro; IPR036890; HATPase_C_sf.DR   InterPro; IPR005467; His_kinase_dom.DR   InterPro; IPR029095; NarX-like_N.DR   InterPro; IPR016380; Sig_transdc_His_kin_NarX/NarQ.DR   InterPro; IPR011712; Sig_transdc_His_kin_sub3_dim/P.DR   Pfam; PF00672; HAMP; 1.DR   Pfam; PF02518; HATPase_c; 1.DR   Pfam; PF07730; HisKA_3; 1.DR   Pfam; PF13675; PilJ; 1.DR   PIRSF; PIRSF003167; STHK_NarX/NarQ; 1.DR   SMART; SM00304; HAMP; 1.DR   SMART; SM00387; HATPase_c; 1.DR   SUPFAM; SSF55874; SSF55874; 1.DR   PROSITE; PS50885; HAMP; 1.DR   PROSITE; PS50109; HIS_KIN; 1.PE   1: Evidence at protein level;KW   3D-structure; ATP-binding; Cell inner membrane; Cell membrane;KW   Complete proteome; Kinase; Membrane; Nitrate assimilation;KW   Nucleotide-binding; Phosphoprotein; Reference proteome; Transferase;KW   Transmembrane; Transmembrane helix; Two-component regulatory system.FT   CHAIN         1    598       Nitrate/nitrite sensor protein NarX.FT                                /FTId=PRO_0000074812.FT   TOPO_DOM      1     14       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM     15     37       Helical. {ECO:0000255}.FT   TOPO_DOM     38    151       Periplasmic. {ECO:0000255}.FT   TRANSMEM    152    174       Helical. {ECO:0000255}.FT   TOPO_DOM    175    598       Cytoplasmic. {ECO:0000255}.FT   DOMAIN      176    228       HAMP. {ECO:0000255|PROSITE-FT                                ProRule:PRU00102}.FT   DOMAIN      393    587       Histidine kinase. {ECO:0000255|PROSITE-FT                                ProRule:PRU00107}.FT   MOD_RES     399    399       Phosphohistidine; by autocatalysis.FT                                {ECO:0000255|PROSITE-ProRule:PRU00107}.FT   CONFLICT     42     52       GSAHAINKAGS -> AAPMRSTKRDA (in Ref. 1 andFT                                5). {ECO:0000305}.FT   CONFLICT    374    374       A -> G (in Ref. 6; AAA24198).FT                                {ECO:0000305}.FT   HELIX        43     62       {ECO:0000244|PDB:3EZH}.FT   HELIX        68     70       {ECO:0000244|PDB:3EZH}.FT   HELIX        71     81       {ECO:0000244|PDB:3EZH}.FT   HELIX        84     93       {ECO:0000244|PDB:3EZH}.FT   HELIX        96    108       {ECO:0000244|PDB:3EZH}.FT   HELIX       110    116       {ECO:0000244|PDB:3EZH}.FT   HELIX       120    122       {ECO:0000244|PDB:3EZH}.FT   HELIX       124    150       {ECO:0000244|PDB:3EZH}.SQ   SEQUENCE   598 AA;  67084 MW;  886BA0FC2C8F3C3E CRC64;     MLKRCLSPLT LVNQVALIVL LSTAIGLAGM AVSGWLVQGV QGSAHAINKA GSLRMQSYRL     LAAVPLSEKD KPLIKEMEQT AFSAELTRAA ERDGQLAQLQ GLQDYWRNEL IPALMRAQNR     ETVSADVSQF VAGLDQLVSG FDRTTEMRIE TVVLVHRVMA VFMALLLVFT IIWLRARLLQ     PWRQLLAMAS AVSHRDFTQR ANISGRNEMA MLGTALNNMS AELAESYAVL EQRVQEKTAG     LEHKNQILSF LWQANRRLHS RAPLCERLSP VLNGLQNLTL LRDIELRVYD TDDEENHQEF     TCQPDMTCDD KGCQLCPRGV LPVGDRGTTL KWRLADSHTQ YGILLATLPQ GRHLSHDQQQ     LVDTLVEQLT ATLALDRHQE RQQQLIVMEE RATIARELHD SIAQSLSCMK MQVSCLQMQG     DALPESSREL LSQIRNELNA SWAQLRELLT TFRLQLTEPG LRPALEASCE EYSAKFGFPV     KLDYQLPPRL VPSHQAIHLL QIAREALSNA LKHSQASEVV VTVAQNDNQV KLTVQDNGCG     VPENAIRSNH YGMIIMRDRA QSLRGDCRVR RRESGGTEVV VTFIPEKTFT DVQGDTHE//\\n',\n",
       " ' ID   MURF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_02019,CC       ECO:0000269|PubMed:2186811}.CC   \\n',\n",
       " ' ID   MUTT_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NADD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NANE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MURQ_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MURR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MUTY_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MSYB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MTR_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MQSA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MSBA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01703, ECO:0000269|PubMed:12119303,CC       ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:8809774}; Multi-CC       pass membrane protein {ECO:0000255|HAMAP-Rule:MF_01703,CC       ECO:0000269|PubMed:8809774}.CC   \\n',\n",
       " ' ID   MTLD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MTLR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MUG_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   LACY_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:2164211}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:2164211}.CC   \\n',\n",
       " ' ID   LAFU_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LDCA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:10428950}.CC   \\n',\n",
       " ' ID   LDHD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LDRC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000250|UniProtKB:P0DPD0}; Single-pass membrane proteinCC       {ECO:0000255}.CC   \\n',\n",
       " ' ID   LPTN_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPXA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   LRP_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   LSRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305};CC       Peripheral membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   LGOR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LIPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   LIPB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00013}.CC   \\n',\n",
       " ' ID   LIVH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   LIVK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm.CC   \\n',\n",
       " ' ID   LOLD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Peripheral membraneCC       protein.CC   \\n',\n",
       " ' ID   LONH_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPFUR_ECOLI  \\n',\n",
       " '\\n',\n",
       " ' ID   LPL_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   LPPY_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPRH_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPXB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPXD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPXK_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPXP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01943, ECO:0000305}; Single-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_01943}.CC   \\n',\n",
       " ' ID   LRHA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LEUC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LEUE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000305}; Multi-passCC       membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   LHGO_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LIT_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000305}; Multi-passCC       membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   LLDD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01559, ECO:0000269|PubMed:18473}; Peripheral membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01559,CC       ECO:0000269|PubMed:18473}.CC   \\n',\n",
       " ' ID   LPLA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   LPOB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:21183073, ECO:0000269|PubMed:21183074}; Lipid-CC       anchor {ECO:0000269|PubMed:21183073, ECO:0000269|PubMed:21183074};CC       Periplasmic side {ECO:0000269|PubMed:21183073,CC       ECO:0000269|PubMed:21183074}. Note=Localizes to the divisome andCC       to the lateral wall.CC   \\n',\n",
       " ' ID   LPTC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01915, ECO:0000269|PubMed:18424520,CC       ECO:0000269|PubMed:20720015}; Single-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_01915, ECO:0000269|PubMed:18424520,CC       ECO:0000269|PubMed:20720015}.CC   \\n',\n",
       " ' ID   MALI_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MALQ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MALY_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MATP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01073,CC       ECO:0000269|PubMed:18984159}. Note=Accumulates in the cell as aCC       discrete focus that colocalizes with the Ter macrodomain.CC   \\n',\n",
       " ' ID   MBHM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.CC   \\n',\n",
       " ' ID   MCBR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MCP1_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:22380631}; Multi-CC       pass membrane protein {ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:22380631}. Note=Found predominantly at cellCC       poles.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; V00373; CAA23676.1; ALT_SEQ; Genomic_DNA.DR   EMBL; U14003; AAA97252.1; -; Genomic_DNA.DR   EMBL; U00096; AAC77311.1; -; Genomic_DNA.DR   EMBL; AP009048; BAE78345.1; -; Genomic_DNA.DR   EMBL; S56952; AAB25802.1; -; Genomic_DNA.DR   PIR; E65250; QRECS.DR   RefSeq; NP_418775.1; NC_000913.3.DR   RefSeq; WP_000919536.1; NZ_LN832404.1.DR   PDB; 1QU7; X-ray; 2.60 A; A/B=294-520.DR   PDB; 2D4U; X-ray; 1.95 A; A/B=25-190.DR   PDB; 3ATP; X-ray; 2.50 A; A/B=25-190.DR   PDB; 3ZX6; X-ray; 2.65 A; A/B=264-551.DR   PDBsum; 1QU7; -.DR   PDBsum; 2D4U; -.DR   PDBsum; 3ATP; -.DR   PDBsum; 3ZX6; -.DR   ProteinModelPortal; P02942; -.DR   SMR; P02942; -.DR   BioGrid; 4261744; 203.DR   DIP; DIP-11046N; -.DR   IntAct; P02942; 4.DR   MINT; P02942; -.DR   STRING; 316407.85677095; -.DR   PaxDb; P02942; -.DR   PRIDE; P02942; -.DR   EnsemblBacteria; AAC77311; AAC77311; b4355.DR   EnsemblBacteria; BAE78345; BAE78345; BAE78345.DR   GeneID; 948884; -.DR   KEGG; ecj:JW4318; -.DR   KEGG; eco:b4355; -.DR   PATRIC; fig|1411691.4.peg.2331; -.DR   EchoBASE; EB1027; -.DR   EcoGene; EG11034; tsr.DR   eggNOG; ENOG4105C8Q; Bacteria.DR   eggNOG; COG0840; LUCA.DR   HOGENOM; HOG000148074; -.DR   InParanoid; P02942; -.DR   KO; K05874; -.DR   OMA; HMQGELV; -.DR   PhylomeDB; P02942; -.DR   BioCyc; EcoCyc:TSR-MONOMER; -.DR   EvolutionaryTrace; P02942; -.DR   PRO; PR:P02942; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0005887; C:integral component of plasma membrane; IDA:EcoCyc.DR   GO; GO:0098561; C:methyl accepting chemotaxis protein complex; IMP:CAFA.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   GO; GO:0043424; F:protein histidine kinase binding; IDA:EcoCyc.DR   GO; GO:0042803; F:protein homodimerization activity; IMP:CAFA.DR   GO; GO:0004888; F:transmembrane signaling receptor activity; IMP:EcoCyc.DR   GO; GO:0048870; P:cell motility; IMP:CACAO.DR   GO; GO:0071230; P:cellular response to amino acid stimulus; IMP:CAFA.DR   GO; GO:0006935; P:chemotaxis; IMP:CACAO.DR   GO; GO:0009593; P:detection of chemical stimulus; IMP:CAFA.DR   GO; GO:0051260; P:protein homooligomerization; IMP:CAFA.DR   GO; GO:0043113; P:receptor clustering; IMP:CAFA.DR   GO; GO:1902021; P:regulation of bacterial-type flagellum-dependent cell motility; IMP:CAFA.DR   GO; GO:0050920; P:regulation of chemotaxis; IMP:CAFA.DR   GO; GO:0032110; P:regulation of protein histidine kinase activity; IMP:CAFA.DR   GO; GO:0043200; P:response to amino acid; IMP:EcoCyc.DR   GO; GO:0007172; P:signal complex assembly; IMP:CAFA.DR   GO; GO:0007165; P:signal transduction; IMP:CACAO.DR   CDD; cd06225; HAMP; 1.DR   CDD; cd00181; Tar_Tsr_LBD; 1.DR   InterPro; IPR035440; 4HB_MCP_dom_sf.DR   InterPro; IPR004090; Chemotax_Me-accpt_rcpt.DR   InterPro; IPR004091; Chemotax_Me-accpt_rcpt_Me-site.DR   InterPro; IPR003660; HAMP_dom.DR   InterPro; IPR004089; MCPsignal_dom.DR   InterPro; IPR003122; Tar_rcpt_lig-bd.DR   Pfam; PF00672; HAMP; 1.DR   Pfam; PF00015; MCPsignal; 1.DR   Pfam; PF02203; TarH; 1.DR   PRINTS; PR00260; CHEMTRNSDUCR.DR   SMART; SM00304; HAMP; 1.DR   SMART; SM00283; MA; 1.DR   SMART; SM00319; TarH; 1.DR   SUPFAM; SSF47170; SSF47170; 1.DR   PROSITE; PS00538; CHEMOTAXIS_TRANSDUC_1; 1.DR   PROSITE; PS50111; CHEMOTAXIS_TRANSDUC_2; 1.DR   PROSITE; PS50885; HAMP; 1.PE   1: Evidence at protein level;KW   3D-structure; Cell inner membrane; Cell membrane; Chemotaxis;KW   Complete proteome; Direct protein sequencing; Membrane; Methylation;KW   Reference proteome; Transducer; Transmembrane; Transmembrane helix.FT   CHAIN         1    551       Methyl-accepting chemotaxis protein I.FT                                /FTId=PRO_0000110537.FT   TOPO_DOM      1      6       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM      7     30       Helical. {ECO:0000255}.FT   TOPO_DOM     31    190       Periplasmic. {ECO:0000255}.FT   TRANSMEM    191    210       Helical. {ECO:0000255}.FT   TOPO_DOM    211    551       Cytoplasmic. {ECO:0000255}.FT   DOMAIN      216    268       HAMP. {ECO:0000255|PROSITE-FT                                ProRule:PRU00102}.FT   DOMAIN      273    502       Methyl-accepting transducer.FT                                {ECO:0000255|PROSITE-ProRule:PRU00284}.FT   REGION       64     73       The 3 Arg may form a positively chargedFT                                pocket, which binds the alpha-carboxylFT                                group of the attractant AA.FT   MOD_RES     297    297       Glutamate methyl ester (Gln).FT                                {ECO:0000269|PubMed:2033064,FT                                ECO:0000269|PubMed:6402709}.FT   MOD_RES     304    304       Glutamate methyl ester (Glu).FT                                {ECO:0000269|PubMed:2033064,FT                                ECO:0000269|PubMed:6402709}.FT   MOD_RES     311    311       Glutamate methyl ester (Gln).FT                                {ECO:0000269|PubMed:2033064,FT                                ECO:0000269|PubMed:6402709}.FT   MOD_RES     493    493       Glutamate methyl ester (Glu).FT                                {ECO:0000269|PubMed:2033064,FT                                ECO:0000269|PubMed:6402709}.FT   MOD_RES     502    502       Glutamate methyl ester (Glu).FT                                {ECO:0000269|PubMed:2033064}.FT   HELIX        25     78       {ECO:0000244|PDB:2D4U}.FT   HELIX        87    110       {ECO:0000244|PDB:2D4U}.FT   HELIX       119    144       {ECO:0000244|PDB:2D4U}.FT   HELIX       148    153       {ECO:0000244|PDB:2D4U}.FT   HELIX       156    186       {ECO:0000244|PDB:2D4U}.FT   HELIX       296    328       {ECO:0000244|PDB:1QU7}.FT   HELIX       332    360       {ECO:0000244|PDB:1QU7}.FT   HELIX       362    388       {ECO:0000244|PDB:1QU7}.FT   HELIX       390    392       {ECO:0000244|PDB:1QU7}.FT   HELIX       393    438       {ECO:0000244|PDB:1QU7}.FT   HELIX       441    448       {ECO:0000244|PDB:1QU7}.FT   TURN        449    451       {ECO:0000244|PDB:1QU7}.FT   HELIX       452    515       {ECO:0000244|PDB:1QU7}.SQ   SEQUENCE   551 AA;  59443 MW;  E8DB026029989B0F CRC64;     MLKRIKIVTS LLLVLAVFGL LQLTSGGLFF NALKNDKENF TVLQTIRQQQ STLNGSWVAL     LQTRNTLNRA GIRYMMDQNN IGSGSTVAEL MESASISLKQ AEKNWADYEA LPRDPRQSTA     AAAEIKRNYD IYHNALAELI QLLGAGKINE FFDQPTQGYQ DGFEKQYVAY MEQNDRLHDI     AVSDNNASYS QAMWILVGVM IVVLAVIFAV WFGIKASLVA PMNRLIDSIR HIAGGDLVKP     IEVDGSNEMG QLAESLRHMQ GELMRTVGDV RNGANAIYSG ASEIATGNND LSSRTEQQAA     SLEETAASME QLTATVKQNA ENARQASHLA LSASETAQRG GKVVDNVVQT MRDISTSSQK     IADIISVIDG IAFQTNILAL NAAVEAARAG EQGRGFAVVA GEVRNLAQRS AQAAREIKSL     IEDSVGKVDV GSTLVESAGE TMAEIVSAVT RVTDIMGEIA SASDEQSRGI DQVGLAVAEM     DRVTQQNAAL VEESAAAAAA LEEQASRLTE AVAVFRIQQQ QRETSAVVKT VTPAAPRKMA     VADSEENWET F//\\n',\n",
       " ' ID   MCP2_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:22380631}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:22380631}. Note=Found predominantly at cellCC       poles.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; J01705; AAA23566.1; -; Genomic_DNA.DR   EMBL; U00096; AAC74956.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA15702.1; -; Genomic_DNA.DR   PIR; F64951; QRECM4.DR   RefSeq; NP_416400.1; NC_000913.3.DR   RefSeq; WP_001297437.1; NZ_LN832404.1.DR   PDB; 2ASR; X-ray; 2.30 A; A=38-179.DR   PDB; 2L9G; NMR; -; A=214-232.DR   PDB; 4Z9H; X-ray; 1.45 A; A/B=26-193.DR   PDB; 4Z9I; X-ray; 1.57 A; A/B=26-193.DR   PDB; 4Z9J; X-ray; 1.78 A; A/B=26-193.DR   PDBsum; 2ASR; -.DR   PDBsum; 2L9G; -.DR   PDBsum; 4Z9H; -.DR   PDBsum; 4Z9I; -.DR   PDBsum; 4Z9J; -.DR   ProteinModelPortal; P07017; -.DR   SMR; P07017; -.DR   BioGrid; 4260381; 319.DR   DIP; DIP-10956N; -.DR   IntAct; P07017; 7.DR   STRING; 316385.ECDH10B_2027; -.DR   MoonProt; P07017; -.DR   PaxDb; P07017; -.DR   PRIDE; P07017; -.DR   EnsemblBacteria; AAC74956; AAC74956; b1886.DR   EnsemblBacteria; BAA15702; BAA15702; BAA15702.DR   GeneID; 946399; -.DR   KEGG; ecj:JW1875; -.DR   KEGG; eco:b1886; -.DR   PATRIC; fig|511145.12.peg.1967; -.DR   EchoBASE; EB0981; -.DR   EcoGene; EG10988; tar.DR   eggNOG; ENOG4105C8Q; Bacteria.DR   eggNOG; COG0840; LUCA.DR   HOGENOM; HOG000148074; -.DR   InParanoid; P07017; -.DR   KO; K05875; -.DR   OMA; GDDANWE; -.DR   PhylomeDB; P07017; -.DR   BioCyc; EcoCyc:TAR-MONOMER; -.DR   EvolutionaryTrace; P07017; -.DR   PRO; PR:P07017; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0005887; C:integral component of plasma membrane; IDA:CAFA.DR   GO; GO:0098561; C:methyl accepting chemotaxis protein complex; IDA:CAFA.DR   GO; GO:0043424; F:protein histidine kinase binding; IDA:EcoCyc.DR   GO; GO:0042803; F:protein homodimerization activity; IDA:CAFA.DR   GO; GO:0004871; F:signal transducer activity; IMP:EcoCyc.DR   GO; GO:0004888; F:transmembrane signaling receptor activity; IMP:EcoCyc.DR   GO; GO:0071230; P:cellular response to amino acid stimulus; IMP:CAFA.DR   GO; GO:0006935; P:chemotaxis; IMP:EcoCyc.DR   GO; GO:0009593; P:detection of chemical stimulus; IDA:CAFA.DR   GO; GO:1901875; P:positive regulation of post-translational protein modification; IMP:CAFA.DR   GO; GO:0051260; P:protein homooligomerization; IMP:CAFA.DR   GO; GO:0050920; P:regulation of chemotaxis; IDA:CAFA.DR   GO; GO:0007172; P:signal complex assembly; IDA:CAFA.DR   CDD; cd06225; HAMP; 1.DR   CDD; cd00181; Tar_Tsr_LBD; 1.DR   InterPro; IPR035440; 4HB_MCP_dom_sf.DR   InterPro; IPR004090; Chemotax_Me-accpt_rcpt.DR   InterPro; IPR004091; Chemotax_Me-accpt_rcpt_Me-site.DR   InterPro; IPR003660; HAMP_dom.DR   InterPro; IPR004089; MCPsignal_dom.DR   InterPro; IPR003122; Tar_rcpt_lig-bd.DR   Pfam; PF00672; HAMP; 1.DR   Pfam; PF00015; MCPsignal; 1.DR   Pfam; PF02203; TarH; 1.DR   PRINTS; PR00260; CHEMTRNSDUCR.DR   SMART; SM00304; HAMP; 1.DR   SMART; SM00283; MA; 1.DR   SMART; SM00319; TarH; 1.DR   SUPFAM; SSF47170; SSF47170; 1.DR   PROSITE; PS00538; CHEMOTAXIS_TRANSDUC_1; 1.DR   PROSITE; PS50111; CHEMOTAXIS_TRANSDUC_2; 1.DR   PROSITE; PS50885; HAMP; 1.PE   1: Evidence at protein level;KW   3D-structure; Cell inner membrane; Cell membrane; Chemotaxis;KW   Complete proteome; Membrane; Methylation; Reference proteome;KW   Transducer; Transmembrane; Transmembrane helix.FT   CHAIN         1    553       Methyl-accepting chemotaxis protein II.FT                                /FTId=PRO_0000110538.FT   TOPO_DOM      1      6       Cytoplasmic.FT   TRANSMEM      7     33       Helical. {ECO:0000255}.FT   TOPO_DOM     34    190       Periplasmic.FT   TRANSMEM    191    211       Helical. {ECO:0000255}.FT   TOPO_DOM    212    553       Cytoplasmic.FT   DOMAIN      214    266       HAMP. {ECO:0000255|PROSITE-FT                                ProRule:PRU00102}.FT   DOMAIN      271    500       Methyl-accepting transducer.FT                                {ECO:0000255|PROSITE-ProRule:PRU00284}.FT   REGION       64     73       The 3 Arg may form a positively chargedFT                                pocket, which binds the alpha-carboxylFT                                group of the attractant AA.FT   MOD_RES     295    295       Glutamate methyl ester (Gln).FT                                {ECO:0000250|UniProtKB:P02942}.FT   MOD_RES     302    302       Glutamate methyl ester (Glu).FT                                {ECO:0000250|UniProtKB:P02942}.FT   MOD_RES     309    309       Glutamate methyl ester (Gln).FT                                {ECO:0000250|UniProtKB:P02942}.FT   MOD_RES     491    491       Glutamate methyl ester (Glu).FT                                {ECO:0000250|UniProtKB:P02942}.FT   MOD_RES     500    500       Glutamate methyl ester (Glu).FT                                {ECO:0000250|UniProtKB:P02942}.FT   CONFLICT    164    164       A -> R (in Ref. 1; AAA23566).FT                                {ECO:0000305}.FT   HELIX        37     74       {ECO:0000244|PDB:4Z9H}.FT   HELIX        81     85       {ECO:0000244|PDB:4Z9H}.FT   HELIX        86    109       {ECO:0000244|PDB:4Z9H}.FT   HELIX       114    116       {ECO:0000244|PDB:4Z9H}.FT   HELIX       117    142       {ECO:0000244|PDB:4Z9H}.FT   HELIX       146    151       {ECO:0000244|PDB:4Z9H}.FT   HELIX       154    182       {ECO:0000244|PDB:4Z9H}.FT   TURN        215    219       {ECO:0000244|PDB:2L9G}.FT   HELIX       220    226       {ECO:0000244|PDB:2L9G}.FT   TURN        227    230       {ECO:0000244|PDB:2L9G}.SQ   SEQUENCE   553 AA;  59944 MW;  B8DC7F2229CE7DC8 CRC64;     MINRIRVVTL LVMVLGVFAL LQLISGSLFF SSLHHSQKSF VVSNQLREQQ GELTSTWDLM     LQTRINLSRS AVRMMMDSSN QQSNAKVELL DSARKTLAQA ATHYKKFKSM APLPEMVATS     RNIDEKYKNY YTALTELIDY LDYGNTGAYF AQPTQGMQNA MGEAFAQYAL SSEKLYRDIV     TDNADDYRFA QWQLAVIALV VVLILLVAWY GIRRMLLTPL AKIIAHIREI AGGNLANTLT     IDGRSEMGDL AQSVSHMQRS LTDTVTHVRE GSDAIYAGTR EIAAGNTDLS SRTEQQASAL     EETAASMEQL TATVKQNADN ARQASQLAQS ASDTAQHGGK VVDGVVKTMH EIADSSKKIA     DIISVIDGIA FQTNILALNA AVEAARAGEQ GRGFAVVAGE VRNLASRSAQ AAKEIKALIE     DSVSRVDTGS VLVESAGETM NNIVNAVTRV TDIMGEIASA SDEQSRGIDQ VALAVSEMDR     VTQQNASLVQ ESAAAAAALE EQASRLTQAV SAFRLAASPL TNKPQTPSRP ASEQPPAQPR     LRIAEQDPNW ETF//\\n',\n",
       " ' ID   MCP4_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:22380631}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:22380631}. Note=Found predominantly at cellCC       poles.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; J01705; AAA23567.1; -; Genomic_DNA.DR   EMBL; U00096; AAC74955.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA15701.1; -; Genomic_DNA.DR   PIR; E64951; QRECM2.DR   RefSeq; NP_416399.1; NC_000913.3.DR   RefSeq; WP_000483239.1; NZ_LN832404.1.DR   ProteinModelPortal; P07018; -.DR   SMR; P07018; -.DR   BioGrid; 4262072; 172.DR   DIP; DIP-10955N; -.DR   IntAct; P07018; 2.DR   STRING; 316385.ECDH10B_2026; -.DR   PaxDb; P07018; -.DR   PRIDE; P07018; -.DR   EnsemblBacteria; AAC74955; AAC74955; b1885.DR   EnsemblBacteria; BAA15701; BAA15701; BAA15701.DR   GeneID; 946397; -.DR   KEGG; ecj:JW1874; -.DR   KEGG; eco:b1885; -.DR   PATRIC; fig|1411691.4.peg.362; -.DR   EchoBASE; EB0980; -.DR   EcoGene; EG10987; tap.DR   eggNOG; ENOG4105C8Q; Bacteria.DR   eggNOG; COG0840; LUCA.DR   HOGENOM; HOG000148074; -.DR   InParanoid; P07018; -.DR   KO; K05877; -.DR   OMA; QATWLEN; -.DR   PhylomeDB; P07018; -.DR   BioCyc; EcoCyc:TAP-MONOMER; -.DR   PRO; PR:P07018; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0016021; C:integral component of membrane; IEA:UniProtKB-KW.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   GO; GO:0004871; F:signal transducer activity; IMP:EcoCyc.DR   GO; GO:0004888; F:transmembrane signaling receptor activity; IMP:EcoCyc.DR   GO; GO:0006935; P:chemotaxis; IMP:EcoCyc.DR   CDD; cd06225; HAMP; 1.DR   CDD; cd00181; Tar_Tsr_LBD; 1.DR   InterPro; IPR035440; 4HB_MCP_dom_sf.DR   InterPro; IPR004090; Chemotax_Me-accpt_rcpt.DR   InterPro; IPR004091; Chemotax_Me-accpt_rcpt_Me-site.DR   InterPro; IPR003660; HAMP_dom.DR   InterPro; IPR004089; MCPsignal_dom.DR   InterPro; IPR003122; Tar_rcpt_lig-bd.DR   Pfam; PF00672; HAMP; 1.DR   Pfam; PF00015; MCPsignal; 1.DR   Pfam; PF02203; TarH; 1.DR   PRINTS; PR00260; CHEMTRNSDUCR.DR   SMART; SM00304; HAMP; 1.DR   SMART; SM00283; MA; 1.DR   SMART; SM00319; TarH; 1.DR   SUPFAM; SSF47170; SSF47170; 1.DR   PROSITE; PS00538; CHEMOTAXIS_TRANSDUC_1; 1.DR   PROSITE; PS50111; CHEMOTAXIS_TRANSDUC_2; 1.DR   PROSITE; PS50885; HAMP; 1.PE   1: Evidence at protein level;KW   Cell inner membrane; Cell membrane; Chemotaxis; Complete proteome;KW   Membrane; Methylation; Reference proteome; Transducer; Transmembrane;KW   Transmembrane helix.FT   CHAIN         1    533       Methyl-accepting chemotaxis protein IV.FT                                /FTId=PRO_0000110541.FT   TOPO_DOM      1      6       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM      7     33       Helical. {ECO:0000255}.FT   TOPO_DOM     34    188       Periplasmic. {ECO:0000255}.FT   TRANSMEM    189    209       Helical. {ECO:0000255}.FT   TOPO_DOM    210    533       Cytoplasmic. {ECO:0000255}.FT   DOMAIN      212    264       HAMP. {ECO:0000255|PROSITE-FT                                ProRule:PRU00102}.FT   DOMAIN      269    498       Methyl-accepting transducer.FT                                {ECO:0000255|PROSITE-ProRule:PRU00284}.FT   MOD_RES     293    293       Glutamate methyl ester (Gln).FT                                {ECO:0000250|UniProtKB:P07017}.FT   MOD_RES     300    300       Glutamate methyl ester (Gln).FT                                {ECO:0000250|UniProtKB:P05704}.FT   MOD_RES     307    307       Glutamate methyl ester (Gln).FT                                {ECO:0000250|UniProtKB:P05704}.FT   MOD_RES     489    489       Glutamate methyl ester (Glu).FT                                {ECO:0000250|UniProtKB:P05704}.FT   CONFLICT    335    335       A -> G (in Ref. 1; AAA23567).FT                                {ECO:0000305}.FT   CONFLICT    503    503       H -> R (in Ref. 1; AAA23567).FT                                {ECO:0000305}.FT   CONFLICT    527    533       QIAPVVS -> TNCASGILK (in Ref. 1;FT                                AAA23567). {ECO:0000305}.SQ   SEQUENCE   533 AA;  57512 MW;  632570BE1E45DA38 CRC64;     MFNRIRISTT LFLILILCGI LQIGSNGMSF WAFRDDLQRL NQVEQSNQQR AALAQTRAVM     LQASTALNKA GTLTALSYPA DDIKTLMTTA RASLTQSTTL FKSFMAMTAG NEHVRGLQKE     TEKSFARWHN DLEHQATWLE SNQLSDFLTA PVQGSQNAFD VNFEAWQLEI NHVLEAASAQ     SQRNYQISAL VFISMIIVAA IYISSALWWT RKMIVQPLAI IGSHFDSIAA GNLARPIAVY     GRNEITAIFA SLKTMQQALR GTVSDVRKGS QEMHIGIAEI VAGNNDLSSR TEQQAASLAQ     TAASMEQLTA TVGQNADNAR QASELAKNAA TTAQAGGVQV STMTHTMQEI ATSSQKIGDI     ISVIDGIAFQ TNILALNAAV EAARAGEQGR GFAVVAGEVR NLASRSAQAA KEIKGLIEES     VNRVQQGSKL VNNAAATMID IVSSVTRVND IMGEIASASE EQQRGIEQVA QAVSQMDQVT     QQNASLVEEA AVATEQLANQ ADHLSSRVAV FTLEEHEVAR HESVQLQIAP VVS//\\n',\n",
       " ' ID   LIVG_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LIVM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   LLDP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   LLDR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LOLE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   LPF2_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPID_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPTE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_01186, ECO:0000269|PubMed:20446753,CC       ECO:0000269|PubMed:21257904, ECO:0000269|PubMed:3316191}; Lipid-CC       anchor {ECO:0000255|HAMAP-Rule:MF_01186,CC       ECO:0000269|PubMed:20446753, ECO:0000269|PubMed:21257904,CC       ECO:0000269|PubMed:3316191}. Note=A substantial portion is foundCC       inside the LptD beta-barrel.CC   \\n',\n",
       " ' ID   LPTF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   LSRD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   LDRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:24513967}; Single-pass membrane proteinCC       {ECO:0000305|PubMed:24513967}.CC   \\n',\n",
       " ' ID   LON_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   LPFS_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LPF_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   LPHI_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MAA_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   MALK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01709, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:18456666}; Peripheral membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_01709, ECO:0000269|PubMed:16079137,CC       ECO:0000269|PubMed:18456666}.CC   \\n',\n",
       " ' ID   MAO1_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MAP1_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MARB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LUXS_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MALF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:18456666}; Multi-pass membrane proteinCC       {ECO:0000255|PROSITE-ProRule:PRU00441,CC       ECO:0000269|PubMed:18456666}; Periplasmic sideCC       {ECO:0000269|PubMed:18456666}. Note=A substantial portion of itCC       protrudes into the periplasmic space; inserts in an SRP- and Sec-CC       dependent, YidC-independent fashion into the membrane.CC   \\n',\n",
       " ' ID   MALG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:18456666, ECO:0000269|PubMed:2233678}; Multi-CC       pass membrane protein {ECO:0000255|PROSITE-ProRule:PRU00441,CC       ECO:0000269|PubMed:18456666, ECO:0000269|PubMed:2233678}.CC   \\n',\n",
       " ' ID   MASY_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MCBA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   MCP3_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996, ECO:0000269|PubMed:22380631}; Multi-CC       pass membrane protein {ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:22380631}. Note=Found predominantly at cellCC       poles.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; K02073; AAA81329.2; -; Genomic_DNA.DR   EMBL; U00096; AAC74503.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA15044.1; -; Genomic_DNA.DR   PIR; H64893; QREC3M.DR   RefSeq; NP_415938.1; NC_000913.3.DR   RefSeq; WP_001098559.1; NZ_LN832404.1.DR   ProteinModelPortal; P05704; -.DR   SMR; P05704; -.DR   BioGrid; 4261384; 396.DR   DIP; DIP-11027N; -.DR   IntAct; P05704; 5.DR   STRING; 316385.ECDH10B_1549; -.DR   PaxDb; P05704; -.DR   PRIDE; P05704; -.DR   EnsemblBacteria; AAC74503; AAC74503; b1421.DR   EnsemblBacteria; BAA15044; BAA15044; BAA15044.DR   GeneID; 945995; -.DR   KEGG; ecj:JW1417; -.DR   KEGG; eco:b1421; -.DR   PATRIC; fig|1411691.4.peg.849; -.DR   EchoBASE; EB1011; -.DR   EcoGene; EG11018; trg.DR   eggNOG; ENOG4105C8Q; Bacteria.DR   eggNOG; COG0840; LUCA.DR   HOGENOM; HOG000148074; -.DR   InParanoid; P05704; -.DR   KO; K05876; -.DR   OMA; LQPMLKY; -.DR   PhylomeDB; P05704; -.DR   BioCyc; EcoCyc:TRG-MONOMER; -.DR   PRO; PR:P05704; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0005887; C:integral component of plasma membrane; IMP:CAFA.DR   GO; GO:0098561; C:methyl accepting chemotaxis protein complex; IMP:CAFA.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   GO; GO:0042803; F:protein homodimerization activity; IMP:CAFA.DR   GO; GO:0004888; F:transmembrane signaling receptor activity; IMP:EcoCyc.DR   GO; GO:0006935; P:chemotaxis; IMP:EcoCyc.DR   GO; GO:0007165; P:signal transduction; IMP:EcoCyc.DR   CDD; cd06225; HAMP; 1.DR   CDD; cd00181; Tar_Tsr_LBD; 1.DR   InterPro; IPR035440; 4HB_MCP_dom_sf.DR   InterPro; IPR004090; Chemotax_Me-accpt_rcpt.DR   InterPro; IPR004091; Chemotax_Me-accpt_rcpt_Me-site.DR   InterPro; IPR003660; HAMP_dom.DR   InterPro; IPR004089; MCPsignal_dom.DR   InterPro; IPR003122; Tar_rcpt_lig-bd.DR   Pfam; PF00672; HAMP; 1.DR   Pfam; PF00015; MCPsignal; 1.DR   Pfam; PF02203; TarH; 1.DR   PRINTS; PR00260; CHEMTRNSDUCR.DR   SMART; SM00304; HAMP; 1.DR   SMART; SM00283; MA; 1.DR   SMART; SM00319; TarH; 1.DR   SUPFAM; SSF47170; SSF47170; 1.DR   PROSITE; PS00538; CHEMOTAXIS_TRANSDUC_1; 1.DR   PROSITE; PS50111; CHEMOTAXIS_TRANSDUC_2; 1.DR   PROSITE; PS50885; HAMP; 1.PE   1: Evidence at protein level;KW   Cell inner membrane; Cell membrane; Chemotaxis; Complete proteome;KW   Membrane; Methylation; Reference proteome; Transducer; Transmembrane;KW   Transmembrane helix.FT   CHAIN         1    546       Methyl-accepting chemotaxis protein III.FT                                /FTId=PRO_0000110540.FT   TOPO_DOM      1     23       Cytoplasmic. {ECO:0000255}.FT   TRANSMEM     24     44       Helical. {ECO:0000255}.FT   TOPO_DOM     45    201       Periplasmic. {ECO:0000255}.FT   TRANSMEM    202    222       Helical. {ECO:0000255}.FT   TOPO_DOM    223    546       Cytoplasmic. {ECO:0000255}.FT   DOMAIN      224    276       HAMP. {ECO:0000255|PROSITE-FT                                ProRule:PRU00102}.FT   DOMAIN      281    510       Methyl-accepting transducer.FT                                {ECO:0000255|PROSITE-ProRule:PRU00284}.FT   MOD_RES     305    305       Glutamate methyl ester (Glu).FT                                {ECO:0000269|PubMed:6300110,FT                                ECO:0000269|PubMed:6374654}.FT   MOD_RES     312    312       Glutamate methyl ester (Gln).FT                                {ECO:0000269|PubMed:6300110,FT                                ECO:0000269|PubMed:6374654}.FT   MOD_RES     319    319       Glutamate methyl ester (Gln).FT                                {ECO:0000269|PubMed:6300110,FT                                ECO:0000269|PubMed:6374654}.FT   MOD_RES     501    501       Glutamate methyl ester (Glu).FT                                {ECO:0000269|PubMed:6300110,FT                                ECO:0000269|PubMed:6374654}.FT   MOD_RES     510    510       Glutamate methyl ester (Glu).FT                                {ECO:0000269|PubMed:6300110}.SQ   SEQUENCE   546 AA;  58899 MW;  FDB40374C0E83E7B CRC64;     MNTTPSQRLG FLHHIRLVPL FACILGGILV LFALSSALAG YFLWQADRDQ RDVTAEIEIR     TGLANSSDFL RSARINMIQA GAASRIAEME AMKRNIAQAE SEIKQSQQGY RAYQNRPVKT     PADEALDTEL NQRFQAYITG MQPMLKYAKN GMFEAIINHE SEQIRPLDNA YTDILNKAVK     IRSTRANQLA ELAHQRTRLG GMFMIGAFVL ALVMTLITFM VLRRIVIRPL QHAAQRIEKI     ASGDLTMNDE PAGRNEIGRL SRHLQQMQHS LGMTVGTVRQ GAEEIYRGTS EISAGNADLS     SRTEEQAAAI EQTAASMEQL TATVKQNADN AHHASKLAQE ASIKASDGGQ TVSGVVKTMG     AISTSSKKIS EITAVINSIA FQTNILALNA AVEAARAGEQ GRGFAVVASE VRTLASRSAQ     AAKEIEGLIS ESVRLIDLGS DEVATAGKTM STIVDAVASV THIMQEIAAA SDEQSRGITQ     VSQAISEMDK VTQQNASLVE EASAAAVSLE EQAARLTEAV DVFRLHKHSV SAEPRGAGEP     VSFATV//\\n',\n",
       " ' ID   MCRA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MDTD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MDTH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MDTN_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Single-CC       pass type II membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   KPTA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MAZG_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MBHL_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.CC   \\n',\n",
       " ' ID   MCRB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MDLA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000255|PROSITE-ProRule:PRU00441,CC       ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   MDLB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MDTG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Multi-CC       pass membrane protein {ECO:0000305}.CC   \\n',\n",
       " ' ID   MDTJ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MDTM_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MDTP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000305}; Lipid-CC       anchor {ECO:0000255|PROSITE-ProRule:PRU00303}.CC   \\n',\n",
       " ' ID   MENB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   LSRB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   MACA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:11544226, ECO:0000269|PubMed:17214741};CC       Single-pass membrane protein {ECO:0000269|PubMed:11544226,CC       ECO:0000269|PubMed:17214741}; Periplasmic sideCC       {ECO:0000269|PubMed:11544226, ECO:0000269|PubMed:17214741}.CC   \\n',\n",
       " ' ID   MBHS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Single-pass type ICC       membrane protein; Periplasmic side.CC   \\n',\n",
       " ' ID   MBIA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MDAB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NANA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NANC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane.CC   \\n',\n",
       " ' ID   NANK_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NARJ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:9305880}.CC   \\n',\n",
       " ' ID   NARV_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; X17110; CAA34967.1; -; Genomic_DNA.DR   EMBL; U00096; AAC74547.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA15102.1; -; Genomic_DNA.DR   PIR; S11430; S11430.DR   RefSeq; NP_415982.1; NC_000913.3.DR   RefSeq; WP_000617115.1; NZ_LN832404.1.DR   ProteinModelPortal; P0AF32; -.DR   SMR; P0AF32; -.DR   BioGrid; 4263337; 14.DR   STRING; 316385.ECDH10B_1596; -.DR   TCDB; 5.A.3.1.2; the prokaryotic molybdopterin-containing oxidoreductase (pmo) family.DR   PaxDb; P0AF32; -.DR   PRIDE; P0AF32; -.DR   EnsemblBacteria; AAC74547; AAC74547; b1465.DR   EnsemblBacteria; BAA15102; BAA15102; BAA15102.DR   GeneID; 946029; -.DR   KEGG; ecj:JW1460; -.DR   KEGG; eco:b1465; -.DR   PATRIC; fig|1411691.4.peg.803; -.DR   EchoBASE; EB0638; -.DR   EcoGene; EG10644; narV.DR   eggNOG; ENOG4105D0D; Bacteria.DR   eggNOG; COG2181; LUCA.DR   HOGENOM; HOG000237376; -.DR   InParanoid; P0AF32; -.DR   KO; K00374; -.DR   OMA; RYQVVRS; -.DR   PhylomeDB; P0AF32; -.DR   BioCyc; EcoCyc:NARV-MONOMER; -.DR   BioCyc; MetaCyc:NARV-MONOMER; -.DR   PRO; PR:P0AF32; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0016021; C:integral component of membrane; IEA:UniProtKB-KW.DR   GO; GO:0009325; C:nitrate reductase complex; IEA:InterPro.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   GO; GO:0009055; F:electron transfer activity; IBA:GO_Central.DR   GO; GO:0020037; F:heme binding; ISM:EcoCyc.DR   GO; GO:0046872; F:metal ion binding; IEA:UniProtKB-KW.DR   GO; GO:0008940; F:nitrate reductase activity; IDA:EcoCyc.DR   GO; GO:0019645; P:anaerobic electron transport chain; IDA:EcoCyc.DR   GO; GO:0009061; P:anaerobic respiration; IDA:EcoCyc.DR   GO; GO:0042128; P:nitrate assimilation; IEA:UniProtKB-KW.DR   InterPro; IPR023234; NarG-like_domain.DR   InterPro; IPR036197; NarG-like_sf.DR   InterPro; IPR003816; Nitrate_red_gam.DR   Pfam; PF02665; Nitrate_red_gam; 1.DR   SUPFAM; SSF103501; SSF103501; 1.DR   TIGRFAMs; TIGR00351; narI; 1.PE   1: Evidence at protein level;KW   Cell inner membrane; Cell membrane; Complete proteome;KW   Electron transport; Heme; Iron; Membrane; Metal-binding;KW   Nitrate assimilation; Oxidoreductase; Reference proteome;KW   Transmembrane; Transmembrane helix; Transport.FT   CHAIN         1    226       Respiratory nitrate reductase 2 gammaFT                                chain.FT                                /FTId=PRO_0000096731.FT   TOPO_DOM      1      4       Periplasmic. {ECO:0000250}.FT   TRANSMEM      5     30       Helical; Name=1. {ECO:0000250}.FT   TOPO_DOM     31     48       Cytoplasmic. {ECO:0000250}.FT   TRANSMEM     49     71       Helical; Name=2. {ECO:0000250}.FT   TOPO_DOM     72     83       Periplasmic. {ECO:0000250}.FT   TRANSMEM     84    113       Helical; Name=3. {ECO:0000250}.FT   TOPO_DOM    114    125       Cytoplasmic. {ECO:0000250}.FT   TRANSMEM    126    149       Helical; Name=4. {ECO:0000250}.FT   TOPO_DOM    150    183       Periplasmic. {ECO:0000250}.FT   TRANSMEM    184    199       Helical; Name=5. {ECO:0000250}.FT   TOPO_DOM    200    226       Cytoplasmic. {ECO:0000250}.FT   METAL        57     57       Iron (heme B 1 axial ligand).FT                                {ECO:0000250}.FT   METAL        67     67       Iron (heme B 2 axial ligand).FT                                {ECO:0000250}.FT   METAL       188    188       Iron (heme B 2 axial ligand).FT                                {ECO:0000250}.FT   METAL       206    206       Iron (heme B 1 axial ligand).FT                                {ECO:0000250}.SQ   SEQUENCE   226 AA;  26018 MW;  1D5FAC307F3D5B84 CRC64;     MIQYLNVFFY DIYPYICATV FFLGSWLRYD YGQYTWRASS SQMLDKRGMV IWSNLFHIGI     LGIFFGHLFG MLTPHWMYAW FLPVAAKQLM AMVLGGICGV LTLIGGAGLL WRRLTNQRVR     ATSTTPDIII MSILLIQCLL GLSTIPFSAQ YPDGSEMMKL VGWAQSIVTF RGGSSEMLNG     VAFVFRLHLV LGMTIFLLFP FTRLVHVWSA PFEYFTRRYQ IVRSRR//\\n',\n",
       " ' ID   NARY_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Peripheral membrane protein.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; X17110; CAA34965.1; -; Genomic_DNA.DR   EMBL; U00096; AAC74549.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA15104.1; -; Genomic_DNA.DR   PIR; F64899; F64899.DR   RefSeq; NP_415984.1; NC_000913.3.DR   RefSeq; WP_000702535.1; NZ_LN832404.1.DR   ProteinModelPortal; P19318; -.DR   SMR; P19318; -.DR   BioGrid; 4262894; 13.DR   DIP; DIP-10323N; -.DR   IntAct; P19318; 14.DR   STRING; 316385.ECDH10B_1598; -.DR   TCDB; 5.A.3.1.2; the prokaryotic molybdopterin-containing oxidoreductase (pmo) family.DR   PaxDb; P19318; -.DR   PRIDE; P19318; -.DR   EnsemblBacteria; AAC74549; AAC74549; b1467.DR   EnsemblBacteria; BAA15104; BAA15104; BAA15104.DR   GeneID; 946034; -.DR   KEGG; ecj:JW1462; -.DR   KEGG; eco:b1467; -.DR   PATRIC; fig|1411691.4.peg.801; -.DR   EchoBASE; EB0641; -.DR   EcoGene; EG10647; narY.DR   eggNOG; ENOG4108IUE; Bacteria.DR   eggNOG; COG1140; LUCA.DR   HOGENOM; HOG000237353; -.DR   InParanoid; P19318; -.DR   KO; K00371; -.DR   OMA; MAMRHYK; -.DR   PhylomeDB; P19318; -.DR   BioCyc; EcoCyc:NARY-MONOMER; -.DR   BioCyc; MetaCyc:NARY-MONOMER; -.DR   PRO; PR:P19318; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0009325; C:nitrate reductase complex; IDA:EcoCyc.DR   GO; GO:0005886; C:plasma membrane; IEA:UniProtKB-SubCell.DR   GO; GO:0051538; F:3 iron, 4 sulfur cluster binding; IEA:UniProtKB-KW.DR   GO; GO:0051539; F:4 iron, 4 sulfur cluster binding; IEA:UniProtKB-KW.DR   GO; GO:0009055; F:electron transfer activity; ISM:EcoCyc.DR   GO; GO:0051536; F:iron-sulfur cluster binding; ISM:EcoCyc.DR   GO; GO:0046872; F:metal ion binding; IEA:UniProtKB-KW.DR   GO; GO:0008940; F:nitrate reductase activity; IDA:EcoCyc.DR   GO; GO:0019645; P:anaerobic electron transport chain; IDA:EcoCyc.DR   GO; GO:0009061; P:anaerobic respiration; IDA:EcoCyc.DR   GO; GO:0006974; P:cellular response to DNA damage stimulus; IEP:EcoliWiki.DR   GO; GO:0042128; P:nitrate assimilation; IEA:UniProtKB-KW.DR   CDD; cd10557; NarH_beta-like; 1.DR   Gene3D; 1.10.3650.10; -; 1.DR   InterPro; IPR017896; 4Fe4S_Fe-S-bd.DR   InterPro; IPR029263; Nitr_red_bet_C.DR   InterPro; IPR038262; Nitr_red_bet_C_sf.DR   InterPro; IPR006547; NO3_Rdtase_bsu.DR   Pfam; PF13247; Fer4_11; 1.DR   Pfam; PF14711; Nitr_red_bet_C; 1.DR   TIGRFAMs; TIGR01660; narH; 1.DR   PROSITE; PS51379; 4FE4S_FER_2; 3.PE   1: Evidence at protein level;KW   3Fe-4S; 4Fe-4S; Cell membrane; Complete proteome;KW   Direct protein sequencing; Electron transport; Iron; Iron-sulfur;KW   Membrane; Metal-binding; Nitrate assimilation; Oxidoreductase;KW   Reference proteome; Repeat; Transport.FT   CHAIN         1    514       Respiratory nitrate reductase 2 betaFT                                chain.FT                                /FTId=PRO_0000096734.FT   DOMAIN        7     35       4Fe-4S ferredoxin-type 1.FT                                {ECO:0000255|PROSITE-ProRule:PRU00711}.FT   DOMAIN      174    205       4Fe-4S ferredoxin-type 2.FT                                {ECO:0000255|PROSITE-ProRule:PRU00711}.FT   DOMAIN      207    236       4Fe-4S ferredoxin-type 3.FT                                {ECO:0000255|PROSITE-ProRule:PRU00711}.FT   METAL        16     16       Iron-sulfur 1 (4Fe-4S). {ECO:0000250}.FT   METAL        19     19       Iron-sulfur 1 (4Fe-4S). {ECO:0000250}.FT   METAL        22     22       Iron-sulfur 1 (4Fe-4S). {ECO:0000250}.FT   METAL        26     26       Iron-sulfur 2 (4Fe-4S). {ECO:0000250}.FT   METAL       183    183       Iron-sulfur 3 (4Fe-4S). {ECO:0000250}.FT   METAL       186    186       Iron-sulfur 3 (4Fe-4S). {ECO:0000250}.FT   METAL       191    191       Iron-sulfur 3 (4Fe-4S). {ECO:0000250}.FT   METAL       195    195       Iron-sulfur 4 (3Fe-4S). {ECO:0000250}.FT   METAL       216    216       Iron-sulfur 4 (3Fe-4S). {ECO:0000250}.FT   METAL       222    222       Iron-sulfur 4 (3Fe-4S). {ECO:0000250}.FT   METAL       226    226       Iron-sulfur 3 (4Fe-4S). {ECO:0000250}.FT   METAL       243    243       Iron-sulfur 2 (4Fe-4S). {ECO:0000250}.FT   METAL       246    246       Iron-sulfur 2 (4Fe-4S). {ECO:0000250}.FT   METAL       258    258       Iron-sulfur 2 (4Fe-4S). {ECO:0000250}.FT   METAL       262    262       Iron-sulfur 1 (4Fe-4S). {ECO:0000250}.FT   CONFLICT    325    326       QR -> HG (in Ref. 1; CAA34965).FT                                {ECO:0000305}.SQ   SEQUENCE   514 AA;  58558 MW;  82D997DB093CD818 CRC64;     MKIRSQVGMV LNLDKCIGCH TCSVTCKNVW TGREGMEYAW FNNVETKPGI GYPKNWEDQE     EWQGGWVRDV NGKIRPRLGN KMGVITKIFA NPVVPQIDDY YEPFTFDYEH LHSAPEGKHI     PTARPRSLID GKRMDKVIWG PNWEELLGGE FEKRARDRNF EAMQKEMYGQ FENTFMMYLP     RLCEHCLNPS CVATCPSGAI YKREEDGIVL IDQDKCRGWR LCISGCPYKK IYFNWKSGKS     EKCIFCYPRI ESGQPTVCSE TCVGRIRYLG VLLYDADRIE EAASTEREVD LYERQCEVFL     DPHDPSVIEE ALKQGIPQNV IDAAQRSPVY KMAMDWKLAL PLHPEYRTLP MVWYVPPLSP     IQSYADAGGL PKSEGVLPAI ESLRIPVQYL ANMLSAGDTG PVLRALKRMM AMRHYMRSQT     VEGVTDTRAI DEVGLSVAQV EEMYRYLAIA NYEDRFVIPT SHREMAGDAF AERNGCGFTF     GDGCHGSDSK FNLFNSSRID AINITEVRDK AEGE//\\n',\n",
       " ' ID   NIFJ_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NIMR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NHAA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NHAB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   NIKB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NINE_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NIRC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   NIRD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NLPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane {ECO:0000305}; Lipid-anchorCC       {ECO:0000305}.CC   \\n',\n",
       " ' ID   NLPD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000305}; Lipid-CC       anchor {ECO:0000255|PROSITE-ProRule:PRU00303}. Note=Localizes atCC       the septal ring. {ECO:0000269|PubMed:19525345}.CC   \\n',\n",
       " ' ID   NRDG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NRDH_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NHAR_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; X04382; CAA27969.1; -; Genomic_DNA.DR   EMBL; L24072; AAA24221.1; -; Genomic_DNA.DR   EMBL; U00096; AAC73131.1; -; Genomic_DNA.DR   EMBL; AP009048; BAE76032.1; -; Genomic_DNA.DR   RefSeq; NP_414561.1; NC_000913.3.DR   RefSeq; WP_000062890.1; NZ_LN832404.1.DR   ProteinModelPortal; P0A9G2; -.DR   SMR; P0A9G2; -.DR   BioGrid; 4259724; 3.DR   IntAct; P0A9G2; 2.DR   STRING; 316385.ECDH10B_0021; -.DR   PaxDb; P0A9G2; -.DR   PRIDE; P0A9G2; -.DR   EnsemblBacteria; AAC73131; AAC73131; b0020.DR   EnsemblBacteria; BAE76032; BAE76032; BAE76032.DR   GeneID; 944757; -.DR   KEGG; ecj:JW0019; -.DR   KEGG; eco:b0020; -.DR   PATRIC; fig|511145.12.peg.17; -.DR   EchoBASE; EB1070; -.DR   EcoGene; EG11078; nhaR.DR   eggNOG; ENOG41063Z6; Bacteria.DR   eggNOG; COG0583; LUCA.DR   HOGENOM; HOG000233515; -.DR   InParanoid; P0A9G2; -.DR   KO; K03717; -.DR   OMA; WLNYHHL; -.DR   PhylomeDB; P0A9G2; -.DR   BioCyc; EcoCyc:PD00474; -.DR   PRO; PR:P0A9G2; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0005737; C:cytoplasm; IEA:UniProtKB-SubCell.DR   GO; GO:0003677; F:DNA binding; IEA:UniProtKB-KW.DR   GO; GO:0001140; F:transcriptional activator activity, bacterial-type RNA polymerase proximal promoter sequence-specific DNA binding; IDA:EcoCyc.DR   GO; GO:2000144; P:positive regulation of DNA-templated transcription, initiation; IMP:EcoCyc.DR   Gene3D; 1.10.10.10; -; 1.DR   InterPro; IPR005119; LysR_subst-bd.DR   InterPro; IPR000847; Tscrpt_reg_HTH_LysR.DR   InterPro; IPR036388; WH-like_DNA-bd_sf.DR   InterPro; IPR036390; WH_DNA-bd_sf.DR   Pfam; PF00126; HTH_1; 1.DR   Pfam; PF03466; LysR_substrate; 1.DR   SUPFAM; SSF46785; SSF46785; 1.DR   PROSITE; PS50931; HTH_LYSR; 1.PE   1: Evidence at protein level;KW   Activator; Complete proteome; Cytoplasm; DNA-binding;KW   Reference proteome; Transcription; Transcription regulation.FT   CHAIN         1    301       Transcriptional activator protein NhaR.FT                                /FTId=PRO_0000105689.FT   DOMAIN        6     63       HTH lysR-type. {ECO:0000255|PROSITE-FT                                ProRule:PRU00253}.FT   DNA_BIND     23     42       H-T-H motif. {ECO:0000255|PROSITE-FT                                ProRule:PRU00253}.FT   MUTAGEN     134    134       E->A: In nhaR1; increases theFT                                transcription of NhaA.FT   CONFLICT     46     47       ER -> DA (in Ref. 1, 2 and 3).FT                                {ECO:0000305}.FT   CONFLICT     50     50       G -> A (in Ref. 1, 2 and 3).FT                                {ECO:0000305}.FT   CONFLICT     58     65       RGLEPSEL -> TWSRTQRA (in Ref. 1 and 3).FT                                {ECO:0000305}.FT   CONFLICT    146    146       D -> E (in Ref. 1, 2 and 3).FT                                {ECO:0000305}.FT   CONFLICT    233    301       FGAMHNAIFVAPTLYAYDFYADKTVVEIGRVENVMEEYHAIFT                                FAERMIQHPAVQRICNTDYSALFSPAVR -> WLQVLLVAMFT                                QMQSSLPQRFMHMTFMPIKLS (in Ref. 1 and 3).FT                                {ECO:0000305}.SQ   SEQUENCE   301 AA;  34284 MW;  E81FAFC4FC4FD2F1 CRC64;     MSMSHINYNH LYYFWHVYKE GSVVGAAEAL YLTPQTITGQ IRALEERLQG KLFKRKGRGL     EPSELGELVY RYADKMFTLS QEMLDIVNYR KESNLLFDVG VADALSKRLV SSVLNAAVVE     GEPIHLRCFE STHEMLLEQL SQHKLDMIIS DCPIDSTQQE GLFSVRIGEC GVSFWCTNPP     PEKPFPACLE ERRLLIPGRR SMLGRKLLNW FNSQGLNVEI LGEFDDAALM KAFGAMHNAI     FVAPTLYAYD FYADKTVVEI GRVENVMEEY HAIFAERMIQ HPAVQRICNT DYSALFSPAV     R//\\n',\n",
       " ' ID   NMPC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane; Multi-pass membraneCC       protein. Note=When expressed in mutant CS384.CC   \\n',\n",
       " ' ID   NORR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NORW_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NQOR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NRDR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NIKA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   NIKC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NIKD_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01711}; Peripheral membrane protein {ECO:0000255|HAMAP-CC       Rule:MF_01711}.CC   \\n',\n",
       " ' ID   NIKE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_01712}; Peripheral membrane protein {ECO:0000255|HAMAP-CC       Rule:MF_01712}.CC   \\n',\n",
       " ' ID   NIRB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NLPE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membraneCC       {ECO:0000269|PubMed:15252048, ECO:0000305|PubMed:7635808}; Lipid-CC       anchor {ECO:0000305|PubMed:15252048, ECO:0000305|PubMed:7635808}.CC   \\n',\n",
       " ' ID   NLPI_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell membrane; Lipid-anchor.CC   \\n',\n",
       " ' ID   NPD_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_01121}.CC   \\n',\n",
       " ' ID   NRFF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   NEMA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NIKR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   NFI_ECOLI    \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   NFRA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; L16945; AAC36849.1; -; Unassigned_DNA.DR   EMBL; U82598; AAB40766.1; -; Genomic_DNA.DR   EMBL; U00096; AAC73669.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA35202.1; -; Genomic_DNA.DR   PIR; B49351; B49351.DR   RefSeq; NP_415100.1; NC_000913.3.DR   RefSeq; WP_000662357.1; NZ_LN832404.1.DR   ProteinModelPortal; P31600; -.DR   BioGrid; 4259890; 194.DR   DIP; DIP-10332N; -.DR   IntAct; P31600; 11.DR   STRING; 316385.ECDH10B_0634; -.DR   PaxDb; P31600; -.DR   PRIDE; P31600; -.DR   EnsemblBacteria; AAC73669; AAC73669; b0568.DR   EnsemblBacteria; BAA35202; BAA35202; BAA35202.DR   GeneID; 944741; -.DR   KEGG; ecj:JW0557; -.DR   KEGG; eco:b0568; -.DR   PATRIC; fig|1411691.4.peg.1706; -.DR   EchoBASE; EB1691; -.DR   EcoGene; EG11740; nfrA.DR   eggNOG; ENOG41090FC; Bacteria.DR   eggNOG; COG0457; LUCA.DR   HOGENOM; HOG000117813; -.DR   KO; K11739; -.DR   OMA; WDHALGE; -.DR   BioCyc; EcoCyc:EG11740-MONOMER; -.DR   PRO; PR:P31600; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0009279; C:cell outer membrane; IEA:UniProtKB-SubCell.DR   Gene3D; 1.25.40.10; -; 2.DR   InterPro; IPR025137; NfrA_C.DR   InterPro; IPR013026; TPR-contain_dom.DR   InterPro; IPR011990; TPR-like_helical_dom_sf.DR   InterPro; IPR019734; TPR_repeat.DR   Pfam; PF13283; NfrA_C; 1.DR   SMART; SM00028; TPR; 2.DR   SUPFAM; SSF48452; SSF48452; 4.DR   PROSITE; PS50005; TPR; 3.DR   PROSITE; PS50293; TPR_REGION; 1.PE   1: Evidence at protein level;KW   Cell outer membrane; Complete proteome; Membrane; Reference proteome;KW   Repeat; Signal; TPR repeat.FT   SIGNAL        1     27       {ECO:0000255}.FT   CHAIN        28    990       Bacteriophage adsorption protein A.FT                                /FTId=PRO_0000021809.FT   REPEAT       81    114       TPR 1.FT   REPEAT      612    645       TPR 2.FT   REPEAT      646    679       TPR 3.SQ   SEQUENCE   990 AA;  111308 MW;  AAA616665D1C2E72 CRC64;     MKENNLNRVI GWSGLLLTSL LSTSALADNI GTSAEELGLS DYRHFVIYPR LDKALKAQKN     NDEATAIREF EYIHQQVPDN IPLTLYLAEA YRHFGHDDRA RLLLEDQLKR HPGDARLERS     LAAIPVEVKS VTTVEELLAQ QKACDAAPTL RCRSEVGQNA LRLAQLPVAR AQLNDATFAA     SPEGKTLRTD LLQRAIYLKQ WSQADTLYNE ARQQNTLSAA ERRQWFDVLL AGQLDDRILA     LQSQGIFTDP QSYITYATAL AYRGEKARLQ HYLIENKPLF TTDAQEKSWL YLLSKYSANP     VQALANYTVQ FADNRQYVVG ATLPVLLKEG QYDAAQKLLA TLPANEMLEE RYAVSVATRN     KAEALRLARL LYQQEPANLT RLDQLTWQLM QNEQSREAAD LLLQRYPFQG DARVSQTLMA     RLASLLESHP YLATPAKVAI LSKPLPLAEQ RQWQSQLPGI ADNCPAIVRL LGDMSPSYDA     AAWNRLAKCY RDTLPGVALY AWLQAEQRQP SAWQHRAVAY QAYQVEDYAT ALAAWQKISL     HDMSNEDLLA AANTAQAAGN GAARDRWLQQ AEKRGLGSNA LYWWLHAQRY IPGQPELALN     DLTRSINIAP SANAYVARAT IYRQRHNVPA AVSDLRAALE LEPNNSNTQA ALGYALWDSG     DIAQSREMLE PAHKGLPDDP ALIRQLAYVN QRLDDMPATQ HYARLVIDDI DNQALITPLT     PEQNQQRFNF RRLHEEVGRR WTFSFDSSIG LRSGAMSTAN NNVGGAAPGK SYRSYGQLEA     EYRIGRNMLL EGDLLSVYSR VFADTGENGV MMPVKNPMSG TGLRWKPLRD QIFFIAVEQQ     LPLNGQNGAS DTMLRASASF FNGGKYSDEW HPNGSGWFAQ NLYLDAAQYI RQDIQAWTAD     YRVSWHQKVA NGQTIEPYAH VQDNGYRDKG TQGAQLGGVG VRWNIWTGET HYDAWPHKVS     LGVEYQHTFK AINQRNGERN NAFLTIGVHW//\\n',\n",
       " ' ID   NNR_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   NDPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm, nucleoid.CC   \\n',\n",
       " ' ID   NEPI_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   NFRB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   -----------------------------------------------------------------------CC   Copyrighted by the UniProt Consortium, see https://www.uniprot.org/termsCC   Distributed under the Creative Commons Attribution-NoDerivs LicenseCC   -----------------------------------------------------------------------DR   EMBL; L16945; AAC36848.1; -; Unassigned_DNA.DR   EMBL; U82598; AAB40767.1; -; Genomic_DNA.DR   EMBL; U00096; AAC73670.1; -; Genomic_DNA.DR   EMBL; AP009048; BAA35203.1; -; Genomic_DNA.DR   PIR; A49351; A49351.DR   RefSeq; NP_415101.1; NC_000913.3.DR   RefSeq; WP_000383932.1; NZ_LN832404.1.DR   ProteinModelPortal; P0AFA5; -.DR   BioGrid; 4262937; 95.DR   DIP; DIP-10333N; -.DR   IntAct; P0AFA5; 2.DR   STRING; 316385.ECDH10B_0635; -.DR   PaxDb; P0AFA5; -.DR   PRIDE; P0AFA5; -.DR   EnsemblBacteria; AAC73670; AAC73670; b0569.DR   EnsemblBacteria; BAA35203; BAA35203; BAA35203.DR   GeneID; 945849; -.DR   KEGG; ecj:JW0558; -.DR   KEGG; eco:b0569; -.DR   PATRIC; fig|1411691.4.peg.1705; -.DR   EchoBASE; EB1690; -.DR   EcoGene; EG11739; nfrB.DR   eggNOG; ENOG410802X; Bacteria.DR   eggNOG; ENOG410YBKS; LUCA.DR   HOGENOM; HOG000117834; -.DR   KO; K11740; -.DR   OMA; KTMHDFP; -.DR   BioCyc; EcoCyc:EG11739-MONOMER; -.DR   PRO; PR:P0AFA5; -.DR   Proteomes; UP000000318; Chromosome.DR   Proteomes; UP000000625; Chromosome.DR   GO; GO:0016021; C:integral component of membrane; IEA:UniProtKB-KW.DR   GO; GO:0005886; C:plasma membrane; IDA:EcoCyc.DR   Gene3D; 3.90.550.10; -; 1.DR   InterPro; IPR001173; Glyco_trans_2-like.DR   InterPro; IPR037257; GSPII_E-like_N_sf.DR   InterPro; IPR007831; GSPII_E_N.DR   InterPro; IPR029044; Nucleotide-diphossugar_trans.DR   Pfam; PF13632; Glyco_trans_2_3; 1.DR   Pfam; PF05157; T2SSE_N; 1.DR   SUPFAM; SSF160246; SSF160246; 1.DR   SUPFAM; SSF53448; SSF53448; 1.PE   4: Predicted;KW   Cell inner membrane; Cell membrane; Complete proteome; Membrane;KW   Reference proteome; Transmembrane; Transmembrane helix.FT   CHAIN         1    745       Bacteriophage adsorption protein B.FT                                /FTId=PRO_0000096796.FT   TRANSMEM      8     28       Helical. {ECO:0000255}.FT   TRANSMEM    362    382       Helical. {ECO:0000255}.FT   TRANSMEM    393    413       Helical. {ECO:0000255}.SQ   SEQUENCE   745 AA;  85319 MW;  B596A5D5B64654C0 CRC64;     MDWLLDVFAT WLYGLKVIAI TLAVIMFISG LDDFFIDVVY WVRRIKRKLS VYRRYPRMSY     RELYKPDEKP LAIMVPAWNE TGVIGNMAEL AATTLDYENY HIFVGTYPND PDTQRDVDEV     CARFPNVHKV VCARPGPTSK ADCLNNVLDA ITQFERSANF AFAGFILHDA EDVISPMELR     LFNYLVERKD LIQIPVYPFE REWTHFTSMT YIDEFSELHG KDVPVREALA GQVPSAGVGT     CFSRRAVTAL LADGDGIAFD VQSLTEDYDI GFRLKEKGMT EIFVRFPVVD EAKEREQRKF     LQHARTSNMI CVREYFPDTF STAVRQKSRW IIGIVFQGFK THKWTSSLTL NYFLWRDRKG     AISNFVSFLA MLVMIQLLLL LAYESLWPDA WHFLSIFSGS AWLMTLLWLN FGLMVNRIVQ     RVIFVTGYYG LTQGLLSVLR LFWGNLINFM ANWRALKQVL QHGDPRRVAW DKTTHDFPSV     TGDTRSLRPL GQILLENQVI TEEQLDTALR NRVEGLRLGG SMLMQGLISA EQLAQALAEQ     NGVAWESIDA WQIPSSLIAE MPASVALHYA VLPLRLENDE LIVGSEDGID PVSLAALTRK     VGRKVRYVIV LRGQIVTGLR HWYARRRGHD PRAMLYNAVQ HQWLTEQQAG EIWRQYVPHQ     FLFAEILTTL GHINRSAINV LLLRHERSSL PLGKFLVTEG VISQETLDRV LTIQRELQVS     MQSLLLKAGL NTEQVAQLES ENEGE//\\n',\n",
       " ' ID   NLPA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Lipid-anchor.CC   \\n',\n",
       " ' ID   NOHD_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOEB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOLR_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOTA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MREB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MRR_ECOLI    \\n',\n",
       " '\\n',\n",
       " ' ID   MSRQ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_01207,CC       ECO:0000305|PubMed:26641313}.CC   \\n',\n",
       " ' ID   MSRA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MTFA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:16855233}.CC   \\n',\n",
       " ' ID   MUKB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm, nucleoidCC       {ECO:0000269|PubMed:11886550, ECO:0000269|PubMed:22380631}.CC       Note=Restricted to the nucleoid region, far from the cell poles.CC   \\n',\n",
       " ' ID   MUKE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm, nucleoid. Note=Restricted to theCC       nucleoid region.CC   \\n',\n",
       " ' ID   MUKF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm, nucleoid. Note=Restricted to theCC       nucleoid region.CC   \\n',\n",
       " ' ID   MURA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_00111,CC       ECO:0000305}.CC   \\n',\n",
       " ' ID   MUTH_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MTNN_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MUTS_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MTGA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00766, ECO:0000269|PubMed:8772200}; Single-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_00766}. Note=Localizes to theCC       cell division site. {ECO:0000269|PubMed:18165305}.CC   \\n',\n",
       " ' ID   MURC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000305}.CC   \\n',\n",
       " ' ID   MURI_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MURJ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_02078, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:23935042}; Multi-pass membrane proteinCC       {ECO:0000255|HAMAP-Rule:MF_02078, ECO:0000269|PubMed:15919996,CC       ECO:0000269|PubMed:23935042}.CC   \\n',\n",
       " ' ID   MSRC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MURG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_00033, ECO:0000269|PubMed:8449890}; Peripheral membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_00033,CC       ECO:0000269|PubMed:8449890}.CC   \\n',\n",
       " ' ID   MUTL_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MLTC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000305}; Lipid-CC       anchor {ECO:0000305}.CC   \\n',\n",
       " ' ID   MLTF_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell outer membrane {ECO:0000255|HAMAP-CC       Rule:MF_02016, ECO:0000269|PubMed:18234673}; Peripheral membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_02016,CC       ECO:0000269|PubMed:18234673}. Note=Attached to the inner leafletCC       of the outer membrane.CC   \\n',\n",
       " ' ID   MLTG_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane {ECO:0000255|HAMAP-CC       Rule:MF_02065, ECO:0000269|PubMed:26507882}; Single-pass membraneCC       protein {ECO:0000255|HAMAP-Rule:MF_02065,CC       ECO:0000269|PubMed:26507882}.CC   \\n',\n",
       " ' ID   MNTS_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:21778229}.CC       Note=May be loosely associated with the inner membrane.CC   \\n',\n",
       " ' ID   MOAA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOAC_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MODA_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:8409926,CC       ECO:0000269|PubMed:8576221, ECO:0000269|PubMed:9545596}.CC   \\n',\n",
       " ' ID   MODC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Peripheral membraneCC       protein.CC   \\n',\n",
       " ' ID   MRAZ_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm, nucleoid {ECO:0000255|HAMAP-CC       Rule:MF_01008, ECO:0000269|PubMed:24659771}.CC   \\n',\n",
       " ' ID   MOCA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MODB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membrane; Multi-pass membraneCC       protein.CC   \\n',\n",
       " ' ID   MODE_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm.CC   \\n',\n",
       " ' ID   MOKB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MOKC_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000250|UniProtKB:P0ACG4}; Single-pass membrane proteinCC       {ECO:0000305}.CC   \\n',\n",
       " ' ID   MOTB_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:2447650}; Single-pass type II membrane proteinCC       {ECO:0000269|PubMed:2447650}. Note=The OmpA-like domain probablyCC       functions to anchor the complex to the cell wall.CC   \\n',\n",
       " ' ID   MPAA_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MRAY_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cell inner membraneCC       {ECO:0000269|PubMed:15919996}; Multi-pass membrane proteinCC       {ECO:0000269|PubMed:15919996}.CC   \\n',\n",
       " ' ID   MSRB_ECOLI   \\n',\n",
       " '\\n',\n",
       " ' ID   MSRP_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Periplasm {ECO:0000269|PubMed:16042411}.CC       Note=Is attached to the inner membrane when interacting with theCC       MsrQ subunit.CC   \\n',\n",
       " ' ID   LSRK_ECOLI   \\n',\n",
       " ' SUBCELLULAR LOCATION: Cytoplasm {ECO:0000255|HAMAP-Rule:MF_02053}.CC   \\n',\n",
       " ' ID   MALT_ECOLI   \\n',\n",
       " ...]"
      ]
     },
     "execution_count": 81,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#add extra line for those lacking localization info:\n",
    "for i,j in enumerate(x_list):\n",
    "    if x_list[i][:6]==' ID   ' and x_list[i-1][:6]==' ID   ':\n",
    "        x_list.insert(i,'\\n')\n",
    "x_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [],
   "source": [
    "#export modified list into a file, need to convert list into string before writing into file:\n",
    "\n",
    "with open('/Users/ShuaiWang/Desktop/Shan_Lab/Ribo_seq/Data_analysis/k12_gene_localtion_everyotherline.txt', 'w') as output:\n",
    "    y=''.join(x_list)\n",
    "    output.write(y)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
